{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import tifffile as tff\n",
    "import pandas as pd\n",
    "import os\n",
    "from Bio import Entrez\n",
    "import openpyxl\n",
    "import xlsxwriter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'C:\\\\Users\\\\Mary\\\\Documents\\\\Shared folder\\\\Evidence Synthesis\\\\Scoping Review\\\\Abstract screening results'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.getcwd()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# directory of stage 1 screening results spreadsheet\n",
    "\n",
    "es_dir = \"C:/Users/Mary/Documents/Shared folder/Evidence Synthesis\"\n",
    "# es_stage1_dir = es_dir + \"/Stage 1 Pubmed Screening/\"\n",
    "# es_stage1_dir\n",
    "es_ScR_dir = es_dir + \"/Scoping Review/\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PMID</th>\n",
       "      <th>Title</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Citation</th>\n",
       "      <th>First Author</th>\n",
       "      <th>Journal/Book</th>\n",
       "      <th>Publication Year</th>\n",
       "      <th>Create Date</th>\n",
       "      <th>PMCID</th>\n",
       "      <th>NIHMS ID</th>\n",
       "      <th>DOI</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>35698308</td>\n",
       "      <td>Structural alterations in cell organelles indu...</td>\n",
       "      <td>Parihar A, Dube A.</td>\n",
       "      <td>Luminescence. 2023 Jul;38(7):1175-1184. doi: 1...</td>\n",
       "      <td>Parihar A</td>\n",
       "      <td>Luminescence</td>\n",
       "      <td>2023</td>\n",
       "      <td>2022/06/14</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1002/bio.4307</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>36617247</td>\n",
       "      <td>Endoplasmic Reticulum Bodies in the Lateral Ro...</td>\n",
       "      <td>Toyooka K, Goto Y, Hashimoto K, Wakazaki M, Sa...</td>\n",
       "      <td>Plant Cell Physiol. 2023 May 15;64(5):461-473....</td>\n",
       "      <td>Toyooka K</td>\n",
       "      <td>Plant Cell Physiol</td>\n",
       "      <td>2023</td>\n",
       "      <td>2023/01/08</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1093/pcp/pcac177</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>36602467</td>\n",
       "      <td>Label-free high-resolution white light quantit...</td>\n",
       "      <td>Tayal S, Tiwari S, Mehta DS.</td>\n",
       "      <td>J Biophotonics. 2023 Apr;16(4):e202200298. doi...</td>\n",
       "      <td>Tayal S</td>\n",
       "      <td>J Biophotonics</td>\n",
       "      <td>2023</td>\n",
       "      <td>2023/01/05</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1002/jbio.202200298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>36702897</td>\n",
       "      <td>Three-dimensional structured illumination micr...</td>\n",
       "      <td>Li X, Wu Y, Su Y, Rey-Suarez I, Matthaeus C, U...</td>\n",
       "      <td>Nat Biotechnol. 2023 Sep;41(9):1307-1319. doi:...</td>\n",
       "      <td>Li X</td>\n",
       "      <td>Nat Biotechnol</td>\n",
       "      <td>2023</td>\n",
       "      <td>2023/01/26</td>\n",
       "      <td>PMC10497409</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1038/s41587-022-01651-1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>36774410</td>\n",
       "      <td>Three-dimensional correlative light and focuse...</td>\n",
       "      <td>Yamanishi H, Iwabuchi T.</td>\n",
       "      <td>J Anat. 2023 Jun;242(6):1012-1028. doi: 10.111...</td>\n",
       "      <td>Yamanishi H</td>\n",
       "      <td>J Anat</td>\n",
       "      <td>2023</td>\n",
       "      <td>2023/02/11</td>\n",
       "      <td>PMC10184541</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1111/joa.13831</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>38705628</td>\n",
       "      <td>Laboratory based correlative cryo-soft X-ray t...</td>\n",
       "      <td>Fahy K, Kapishnikov S, Donnellan M, McEnroe T,...</td>\n",
       "      <td>Methods Cell Biol. 2024;187:293-320. doi: 10.1...</td>\n",
       "      <td>Fahy K</td>\n",
       "      <td>Methods Cell Biol</td>\n",
       "      <td>2024</td>\n",
       "      <td>2024/05/05</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1016/bs.mcb.2024.02.033</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>38709484</td>\n",
       "      <td>A Pipeline for Dynamic Analysis of Mitochondri...</td>\n",
       "      <td>Dhyani V, Chann AS, Giri L, Russell SM, Charnl...</td>\n",
       "      <td>Methods Mol Biol. 2024;2800:167-187. doi: 10.1...</td>\n",
       "      <td>Dhyani V</td>\n",
       "      <td>Methods Mol Biol</td>\n",
       "      <td>2024</td>\n",
       "      <td>2024/05/06</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1007/978-1-0716-3834-7_12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>38756048</td>\n",
       "      <td>Characterization of Optical, Thermal, and Visc...</td>\n",
       "      <td>Rawat S, Trius Béjar J, Wang A.</td>\n",
       "      <td>ACS Appl Bio Mater. 2024 Jun 17;7(6):4029-4038...</td>\n",
       "      <td>Rawat S</td>\n",
       "      <td>ACS Appl Bio Mater</td>\n",
       "      <td>2024</td>\n",
       "      <td>2024/05/17</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1021/acsabm.4c00367</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>38786091</td>\n",
       "      <td>Subcellular Feature-Based Classification of α ...</td>\n",
       "      <td>Deshmukh A, Chang K, Cuala J, Vanslembrouck B,...</td>\n",
       "      <td>Cells. 2024 May 18;13(10):869. doi: 10.3390/ce...</td>\n",
       "      <td>Deshmukh A</td>\n",
       "      <td>Cells</td>\n",
       "      <td>2024</td>\n",
       "      <td>2024/05/24</td>\n",
       "      <td>PMC11119489</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.3390/cells13100869</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>38854505</td>\n",
       "      <td>Pleomorphism in Biological Units of Life: Morp...</td>\n",
       "      <td>Singh P, Mittal A.</td>\n",
       "      <td>ACS Omega. 2024 May 20;9(22):23377-23389. doi:...</td>\n",
       "      <td>Singh P</td>\n",
       "      <td>ACS Omega</td>\n",
       "      <td>2024</td>\n",
       "      <td>2024/06/10</td>\n",
       "      <td>PMC11154962</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10.1021/acsomega.3c10062</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>88 rows × 11 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        PMID                                              Title  \\\n",
       "0   35698308  Structural alterations in cell organelles indu...   \n",
       "1   36617247  Endoplasmic Reticulum Bodies in the Lateral Ro...   \n",
       "2   36602467  Label-free high-resolution white light quantit...   \n",
       "3   36702897  Three-dimensional structured illumination micr...   \n",
       "4   36774410  Three-dimensional correlative light and focuse...   \n",
       "..       ...                                                ...   \n",
       "83  38705628  Laboratory based correlative cryo-soft X-ray t...   \n",
       "84  38709484  A Pipeline for Dynamic Analysis of Mitochondri...   \n",
       "85  38756048  Characterization of Optical, Thermal, and Visc...   \n",
       "86  38786091  Subcellular Feature-Based Classification of α ...   \n",
       "87  38854505  Pleomorphism in Biological Units of Life: Morp...   \n",
       "\n",
       "                                              Authors  \\\n",
       "0                                  Parihar A, Dube A.   \n",
       "1   Toyooka K, Goto Y, Hashimoto K, Wakazaki M, Sa...   \n",
       "2                        Tayal S, Tiwari S, Mehta DS.   \n",
       "3   Li X, Wu Y, Su Y, Rey-Suarez I, Matthaeus C, U...   \n",
       "4                            Yamanishi H, Iwabuchi T.   \n",
       "..                                                ...   \n",
       "83  Fahy K, Kapishnikov S, Donnellan M, McEnroe T,...   \n",
       "84  Dhyani V, Chann AS, Giri L, Russell SM, Charnl...   \n",
       "85                    Rawat S, Trius Béjar J, Wang A.   \n",
       "86  Deshmukh A, Chang K, Cuala J, Vanslembrouck B,...   \n",
       "87                                 Singh P, Mittal A.   \n",
       "\n",
       "                                             Citation First Author  \\\n",
       "0   Luminescence. 2023 Jul;38(7):1175-1184. doi: 1...    Parihar A   \n",
       "1   Plant Cell Physiol. 2023 May 15;64(5):461-473....    Toyooka K   \n",
       "2   J Biophotonics. 2023 Apr;16(4):e202200298. doi...      Tayal S   \n",
       "3   Nat Biotechnol. 2023 Sep;41(9):1307-1319. doi:...         Li X   \n",
       "4   J Anat. 2023 Jun;242(6):1012-1028. doi: 10.111...  Yamanishi H   \n",
       "..                                                ...          ...   \n",
       "83  Methods Cell Biol. 2024;187:293-320. doi: 10.1...       Fahy K   \n",
       "84  Methods Mol Biol. 2024;2800:167-187. doi: 10.1...     Dhyani V   \n",
       "85  ACS Appl Bio Mater. 2024 Jun 17;7(6):4029-4038...      Rawat S   \n",
       "86  Cells. 2024 May 18;13(10):869. doi: 10.3390/ce...   Deshmukh A   \n",
       "87  ACS Omega. 2024 May 20;9(22):23377-23389. doi:...      Singh P   \n",
       "\n",
       "          Journal/Book  Publication Year Create Date        PMCID NIHMS ID  \\\n",
       "0         Luminescence              2023  2022/06/14          NaN      NaN   \n",
       "1   Plant Cell Physiol              2023  2023/01/08          NaN      NaN   \n",
       "2       J Biophotonics              2023  2023/01/05          NaN      NaN   \n",
       "3       Nat Biotechnol              2023  2023/01/26  PMC10497409      NaN   \n",
       "4               J Anat              2023  2023/02/11  PMC10184541      NaN   \n",
       "..                 ...               ...         ...          ...      ...   \n",
       "83   Methods Cell Biol              2024  2024/05/05          NaN      NaN   \n",
       "84    Methods Mol Biol              2024  2024/05/06          NaN      NaN   \n",
       "85  ACS Appl Bio Mater              2024  2024/05/17          NaN      NaN   \n",
       "86               Cells              2024  2024/05/24  PMC11119489      NaN   \n",
       "87           ACS Omega              2024  2024/06/10  PMC11154962      NaN   \n",
       "\n",
       "                             DOI  \n",
       "0               10.1002/bio.4307  \n",
       "1            10.1093/pcp/pcac177  \n",
       "2         10.1002/jbio.202200298  \n",
       "3     10.1038/s41587-022-01651-1  \n",
       "4              10.1111/joa.13831  \n",
       "..                           ...  \n",
       "83    10.1016/bs.mcb.2024.02.033  \n",
       "84  10.1007/978-1-0716-3834-7_12  \n",
       "85        10.1021/acsabm.4c00367  \n",
       "86         10.3390/cells13100869  \n",
       "87      10.1021/acsomega.3c10062  \n",
       "\n",
       "[88 rows x 11 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# load results from Stage 1 screening (title/PubMed blurb)\n",
    "coll_031523 = pd.read_csv(es_ScR_dir+\"ScR_round1_071724.csv\")\n",
    "# \n",
    "coll_031523"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# from BioPython Entrez automatic search.ipynb\n",
    "# takes in id_list from search function output, returns a dictionary with items \n",
    "# PubmedArticle and PubmedBookArticle, which are lists of info about each article entry, including metadata, title, etc\n",
    "\n",
    "def fetch_details(id_list):\n",
    "#     ids = ','.join(id_list)\n",
    "    Entrez.email = 'mariya.mirvis@ucsf.edu'\n",
    "    handle = Entrez.efetch(db='pubmed',\n",
    "                           retmode='xml',\n",
    "                           id=id_list)\n",
    "    results = Entrez.read(handle)\n",
    "    return results\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# perform Entrez fetch on list of PMIDs from 1st round of screening\n",
    "\n",
    "stage1_fetch = fetch_details(coll_031523['PMID'].values)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "DictElement({'Language': ['eng'], 'ELocationID': [StringElement('10.1002/smtd.202300447', attributes={'EIdType': 'doi', 'ValidYN': 'Y'})], 'ArticleDate': [DictElement({'Year': '2023', 'Month': '09', 'Day': '05'}, attributes={'DateType': 'Electronic'})], 'Journal': {'ISSN': StringElement('2366-9608', attributes={'IssnType': 'Electronic'}), 'JournalIssue': DictElement({'Volume': '7', 'Issue': '11', 'PubDate': {'Year': '2023', 'Month': 'Nov'}}, attributes={'CitedMedium': 'Internet'}), 'Title': 'Small methods', 'ISOAbbreviation': 'Small Methods'}, 'ArticleTitle': 'Label-Free Intracellular Multi-Specificity in Yeast Cells by Phase-Contrast Tomographic Flow Cytometry.', 'Pagination': {'StartPage': 'e2300447', 'MedlinePgn': 'e2300447'}, 'Abstract': {'AbstractText': [\"In-flow phase-contrast tomography provides a 3D refractive index of label-free cells in cytometry systems. Its major limitation, as with any quantitative phase imaging approach, is the lack of specificity compared to fluorescence microscopy, thus restraining its huge potentialities in single-cell analysis and diagnostics. Remarkable results in introducing specificity are obtained through artificial intelligence (AI), but only for adherent cells. However, accessing the 3D fluorescence ground truth and obtaining accurate voxel-level co-registration of image pairs for AI training is not viable for high-throughput cytometry. The recent statistical inference approach is a significant step forward for label-free specificity but remains limited to cells' nuclei. Here, a generalized computational strategy based on a self-consistent statistical inference to achieve intracellular multi-specificity is shown. Various subcellular compartments (i.e., nuclei, cytoplasmic vacuoles, the peri-vacuolar membrane area, cytoplasm, vacuole-nucleus contact site) can be identified and characterized quantitatively at different phases of the cells life cycle by using yeast cells as a biological model. Moreover, for the first time, virtual reality is introduced for handling the information content of multi-specificity in single cells. Full fruition is proofed for exploring and interacting with 3D quantitative biophysical parameters of the identified compartments on demand, thus opening the route to a metaverse for 3D microscopy.\"], 'CopyrightInformation': '© 2023 The Authors. Small Methods published by Wiley-VCH GmbH.'}, 'AuthorList': ListElement([DictElement({'Identifier': [StringElement('0000-0003-1956-4976', attributes={'Source': 'ORCID'})], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'CNR-ISASI, Institute of Applied Sciences and Intelligent Systems \"E. Caianiello\", Via Campi Flegrei 34, Pozzuoli, Napoli, 80078, Italy.'}], 'LastName': 'Bianco', 'ForeName': 'Vittorio', 'Initials': 'V'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Molecular Medicine and Medical Biotechnology, University of Naples \"Federico II\", Via S. Pansini 5, Naples, 80131, Italy.'}], 'LastName': \"D'Agostino\", 'ForeName': 'Massimo', 'Initials': 'M'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'CNR-ISASI, Institute of Applied Sciences and Intelligent Systems \"E. Caianiello\", Via Campi Flegrei 34, Pozzuoli, Napoli, 80078, Italy.'}], 'LastName': 'Pirone', 'ForeName': 'Daniele', 'Initials': 'D'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'CNR-ISASI, Institute of Applied Sciences and Intelligent Systems \"E. Caianiello\", Via Campi Flegrei 34, Pozzuoli, Napoli, 80078, Italy.'}], 'LastName': 'Giugliano', 'ForeName': 'Giusy', 'Initials': 'G'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Intelligent Industrial Technologies and Systems for Advanced Manufacturing, National Research Council of Italy, Via Amendola 122/D-O, Bari, 70125, Italy.'}], 'LastName': 'Mosca', 'ForeName': 'Nicola', 'Initials': 'N'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Intelligent Industrial Technologies and Systems for Advanced Manufacturing, National Research Council of Italy, Via Amendola 122/D-O, Bari, 70125, Italy.'}], 'LastName': 'Di Summa', 'ForeName': 'Maria', 'Initials': 'M'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Molecular Medicine and Medical Biotechnology, University of Naples \"Federico II\", Via S. Pansini 5, Naples, 80131, Italy.'}], 'LastName': 'Scerra', 'ForeName': 'Gianluca', 'Initials': 'G'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'CNR-ISASI, Institute of Applied Sciences and Intelligent Systems \"E. Caianiello\", Via Campi Flegrei 34, Pozzuoli, Napoli, 80078, Italy.'}], 'LastName': 'Memmolo', 'ForeName': 'Pasquale', 'Initials': 'P'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'CNR-ISASI, Institute of Applied Sciences and Intelligent Systems \"E. Caianiello\", Via Campi Flegrei 34, Pozzuoli, Napoli, 80078, Italy.'}], 'LastName': 'Miccio', 'ForeName': 'Lisa', 'Initials': 'L'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Department of Molecular Medicine and Medical Biotechnology, University of Naples \"Federico II\", Via S. Pansini 5, Naples, 80131, Italy.'}], 'LastName': 'Russo', 'ForeName': 'Tommaso', 'Initials': 'T'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'Institute of Intelligent Industrial Technologies and Systems for Advanced Manufacturing, National Research Council of Italy, Via Amendola 122/D-O, Bari, 70125, Italy.'}], 'LastName': 'Stella', 'ForeName': 'Ettore', 'Initials': 'E'}, attributes={'ValidYN': 'Y'}), DictElement({'Identifier': [], 'AffiliationInfo': [{'Identifier': [], 'Affiliation': 'CNR-ISASI, Institute of Applied Sciences and Intelligent Systems \"E. Caianiello\", Via Campi Flegrei 34, Pozzuoli, Napoli, 80078, Italy.'}], 'LastName': 'Ferraro', 'ForeName': 'Pietro', 'Initials': 'P'}, attributes={'ValidYN': 'Y'})], attributes={'CompleteYN': 'Y'}), 'PublicationTypeList': [StringElement('Journal Article', attributes={'UI': 'D016428'})]}, attributes={'PubModel': 'Print-Electronic'})"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# abstract = stage1_fetch['PubmedArticle'][39]['MedlineCitation']['Article']['Abstract']['AbstractText']\n",
    "stage1_fetch['PubmedArticle'][39]['MedlineCitation']['Article']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "'Abstract' in stage1_fetch['PubmedArticle'][40]['MedlineCitation']['Article']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "35698308 Title:  Structural alterations in cell organelles induced by photodynamic treatment with chlorin p(6) -histamine conjugate in human oral carcinoma cells probed by 3D fluorescence microscopy\n",
      "Abstract:  [\"We have explored the intracellular cell organelle's structural alterations after photodynamic treatment with chlorin p<sub>6</sub> -histamine conjugate (Cp<sub>6</sub> -his) in human oral cancer cells. Herein, the cells were treated with Cp<sub>6</sub> -his (10\\xa0μm) and counterstained with organelle-specific fluorescence probes to find the site of intracellular localization using confocal microscopy. For photodynamic therapy (PDT), the cells were exposed to ~30\\u2009kJ/m<sup>2</sup> red light (660\\u2009±\\u200920\\u2009nm) to induce ~90% cytotoxicity. We used the three-dimensional (3D) image reconstruction approach to analyze the photodynamic damage to cell organelles. The result showed that Cp<sub>6</sub> -his localized mainly in the endoplasmic reticulum (ER) and lysosomes but not in mitochondria and Golgi apparatus (GA). The 3D model revealed that in necrotic cells, PDT led to extensive fragmentation of ER and fragmentation and swelling of GA as well. Results suggest that the indirect damage to GA occurred due to loss of connection between ER and GA. Moreover, in damaged cells with no sign of necrosis, the perinuclear ER appeared condensed and surrounded by several small clumps at the peripheral region of the cell, and the GA was observed to form a single condensed structure. Since these structural changes were associated with apoptotic cell death, it is suggested that the necrotic and apoptotic death induced by PDT with Cp<sub>6</sub> -his is determined by the severity of damage to ER and indirect damage to GA. The results suggest that the indirect damage to cell organelle apart from the sites of photosensitizer localization and the severity of damage at the organelle level contribute significantly to the mode of cell death in PDT.\"]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell?\n",
      "Stop? (Y/N)N\n",
      "36617247 Title:  Endoplasmic Reticulum Bodies in the Lateral Root Cap Are Involved in the Direct Transport of Beta-Glucosidase to Vacuoles\n",
      "Abstract:  ['Programmed cell death (PCD) in lateral root caps (LRCs) is crucial for maintaining root cap functionality. Endoplasmic reticulum (ER) bodies play important roles in plant immunity and PCD. However, the distribution of ER bodies and their communication with vacuoles in the LRC remain elusive. In this study, we investigated the ultrastructure of LRC cells of wild-type and transgenic Arabidopsis lines using an auto-acquisition transmission electron microscope (TEM) system and high-pressure freezing. Gigapixel-scale high-resolution TEM imaging of the transverse and longitudinal sections of roots followed by three-dimensional imaging identified sausage-shaped structures budding from the ER. These were subsequently identified as ER bodies using GFPh transgenic lines expressing green fluorescent protein (GFP) fused with an ER retention signal (HDEL). Immunogold labeling using an anti-GFP antibody detected GFP signals in the ER bodies and vacuoles. The fusion of ER bodies with vacuoles in LRC cells was identified using correlative light and electron microscopy. Imaging of the root tips of a GFPh transgenic line with a PYK10 promoter revealed the localization of PYK10, a member of the β-glucosidase family with an ER retention signal, in the ER bodies in the inner layer along with a fusion of ER bodies with vacuoles in the middle layer and collapse of vacuoles in the outer layer of the LRC. These findings suggest that ER bodies in LRC directly transport β-glucosidases to the vacuoles, and that a subsequent vacuolar collapse triggered by an unknown mechanism releases protective substances to the growing root tip to protect it from the invaders.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: 3D?\n",
      "Stop? (Y/N)N\n",
      "36602467 Title:  Label-free high-resolution white light quantitative phase nanoscopy system\n",
      "Abstract:  ['We present a high-resolution white light quantitative phase nanoscopy (WLQPN) system that can be utilized to visualize nanoparticles and subcellular features of the biological specimens. The five-phase shifting technique, along with deconvolution, is adopted to obtain super-resolution in phase imaging. The phase shifting technique can provide full detector resolution, making it beneficial as compared to the well-known Fourier analysis method. The Fourier transform method requires minimum angle of <math xmlns=\"http://www.w3.org/1998/Math/MathML\"><mrow><mspace></mspace> <msup><mi>sin</mi> <mrow><mo>-</mo> <mn>1</mn></mrow> </msup> <mfenced><mrow><mn>3</mn> <msub><mi>f</mi> <mi>x</mi></msub> <mi>λ</mi></mrow> </mfenced> </mrow> </math> , where <math xmlns=\"http://www.w3.org/1998/Math/MathML\"> <mrow><msub><mi>f</mi> <mi>x</mi></msub> </mrow> </math> is maximum achievable spatial frequency. It limits the highest achievable resolution to much below the actual diffraction limit of the system. Thus, to obtain a high-resolution phase map of the biological specimen, a two-step process is adopted. First, the phase map is recovered using the five-phase shifting algorithm, with full detector spatial resolution. Second, the complex field is obtained from the recovered phase map and further processed using the Richardson Lucy total variation deconvolution algorithm to obtain super-resolution phase images. The present technique was tested on 1951 USAF resolution chart, 200\\u2009nm polystyrene beads and Escherichia coli bacteria using a 50×, 0.55NA objective lens. The 200\\u2009nm polystyrene beads are visually resolvable and subcellular features of the E. coli bacteria are also observed, suggesting a significant improvement in the resolution.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: bacteria; no relevant data\n",
      "Stop? (Y/N)N\n",
      "36702897 Title:  Three-dimensional structured illumination microscopy with enhanced axial resolution\n",
      "Abstract:  ['The axial resolution of three-dimensional structured illumination microscopy (3D SIM) is limited to ∼300\\u2009nm. Here we present two distinct, complementary methods to improve axial resolution in 3D SIM with minimal or no modification to the optical system. We show that placing a mirror directly opposite the sample enables four-beam interference with higher spatial frequency content than 3D SIM illumination, offering near-isotropic imaging with ∼120-nm lateral and 160-nm axial resolution. We also developed a deep learning method achieving ∼120-nm isotropic resolution. This method can be combined with denoising to facilitate volumetric imaging spanning dozens of timepoints. We demonstrate the potential of these advances by imaging a variety of cellular samples, delineating the nanoscale distribution of vimentin and microtubule filaments, observing the relative positions of caveolar coat proteins and lysosomal markers and visualizing cytoskeletal dynamics within T cells in the early stages of immune synapse formation.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: not whole cell\n",
      "Stop? (Y/N)N\n",
      "36774410 Title:  Three-dimensional correlative light and focused ion beam scanning electron microscopy reveals the distribution and ultrastructure of lanceolate nerve endings surrounding terminal hair follicles in human scalp skin\n",
      "Abstract:  ['Lanceolate nerve endings (LNEs) surrounding hair follicles (HFs) play an important role in detecting hair deflection. Complexes of the LNEs form a palisade-like structure along the longitudinal axis of hair roots in which axons are sandwiched between two processes of terminal Schwann cells (tSCs) at the isthmus of HFs. The structure and molecular mechanism of LNEs in animal sinus hair, pelage, and human vellus hairs have been investigated. Despite the high density of HFs in human scalp skin, the LNEs in human terminal HFs have not been investigated. In this study, we aimed to reveal the distribution and ultrastructure of LNEs in terminal HFs of human scalp skin. Using light-sheet microscopy and immunostaining, the LNEs were observed at one terminal HF but not at the other terminal HFs in the same follicular unit. The ultrastructure of the LNEs of terminal HFs in human scalp skin was characterized using correlated light and electron microscopy (CLEM). Confocal laser microscopy and transmission electron microscopy of serial transverse sections of HFs revealed that LNEs were aligned adjacent to the basal lamina outside the outer root sheath (ORS), at the isthmus of terminal HFs, and adjacent to CD200-positive ORS cells in the upper bulge region. Moreover, axons with abundant mitochondria were sandwiched between tSCs. Three-dimensional CLEM, specifically confocal laser microscopy and focused ion beam scanning electron microscopy, of stained serial transverse sections revealed that LNEs were wrapped with type I and type II tSCs, with the processes protruding from the space between the Schwann cells. Moreover, the ultrastructures of LNEs at miniaturized HFs were similar to those of LNEs at terminal HFs. Preembedding immunoelectron microscopy revealed that Piezo-type mechanosensitive ion channel component 2 (Piezo2), a gated ion channel, was in axons and tSCs and adjacent to the cell membrane of axons and tSCs, suggesting that LNEs function as mechanosensors. The number of LNEs increased as the diameter of the ORS decreased, suggesting that LNEs dynamically adapt to the HF environment as terminal HFs miniaturize into vellus-like hair. These findings will provide insights for investigations of mechanosensory organs, aging, and re-innervation during wound healing.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: tissue scale; not whole-cell\n",
      "Stop? (Y/N)N\n",
      "36805107 Title:  Cryo-plasma FIB/SEM volume imaging of biological specimens\n",
      "Abstract:  [\"Serial focussed ion beam scanning electron microscopy (FIB/SEM) enables imaging and assessment of subcellular structures on the mesoscale (10 nm to 10 µm). When applied to vitrified samples, serial FIB/SEM is also a means to target specific structures in cells and tissues while maintaining constituents' hydration shells for in situ structural biology downstream. However, the application of serial FIB/SEM imaging of non-stained cryogenic biological samples is limited due to low contrast, curtaining, and charging artefacts. We address these challenges using a cryogenic plasma FIB/SEM. We evaluated the choice of plasma ion source and imaging regimes to produce high-quality SEM images of a range of different biological samples. Using an automated workflow we produced three-dimensional volumes of bacteria, human cells, and tissue, and calculated estimates for their resolution, typically achieving 20-50 nm. Additionally, a tag-free localisation tool for regions of interest is needed to drive the application of in situ structural biology towards tissue. The combination of serial FIB/SEM with plasma-based ion sources promises a framework for targeting specific features in bulk-frozen samples (>100 µm) to produce lamellae for cryogenic electron tomography.\"]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: 3D volume of cells - whole? organelles?\n",
      "Stop? (Y/N)N\n",
      "36823335 Title:  Field-dependent deep learning enables high-throughput whole-cell 3D super-resolution imaging\n",
      "Abstract:  ['Single-molecule localization microscopy in a typical wide-field setup has been widely used for investigating subcellular structures with super resolution; however, field-dependent aberrations restrict the field of view (FOV) to only tens of micrometers. Here, we present a deep-learning method for precise localization of spatially variant point emitters (FD-DeepLoc) over a large FOV covering the full chip of a modern sCMOS camera. Using a graphic processing unit-based vectorial point spread function (PSF) fitter, we can fast and accurately model the spatially variant PSF of a high numerical aperture objective in the entire FOV. Combined with deformable mirror-based optimal PSF engineering, we demonstrate high-accuracy three-dimensional single-molecule localization microscopy over a volume of ~180\\u2009×\\u2009180\\u2009×\\u20095\\u2009μm<sup>3</sup>, allowing us to image mitochondria and nuclear pore complexes in entire cells in a single imaging cycle without hardware scanning; a 100-fold increase in throughput compared to the state of the art.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell w/ multi-organelles?\n",
      "Stop? (Y/N)N\n",
      "36869426 Title:  Systematic Transmission Electron Microscopy-Based Identification and 3D Reconstruction of Cellular Degradation Machinery\n",
      "Abstract:  ['Various intracellular degradation organelles, including autophagosomes, lysosomes, and endosomes, work in tandem to perform autophagy, which is crucial for cellular homeostasis. Altered autophagy contributes to the pathophysiology of various diseases, including cancers and metabolic diseases. This paper aims to describe an approach to reproducibly identify and distinguish subcellular structures involved in macroautophagy. Methods are provided that help avoid common pitfalls. How to distinguish between lysosomes, lipid droplets, autolysosomes, autophagosomes, and inclusion bodies are also discussed. These methods use transmission electron microscopy (TEM), which is able to generate nanometer-scale micrographs of cellular degradation components in a fixed sample. Serial block face-scanning electron microscopy is also used to visualize the\\xa03D morphology of degradation machinery using the Amira software. In addition to TEM and 3D reconstruction, other imaging techniques are discussed, such as immunofluorescence and immunogold labeling, which can be used to classify cellular organelles, reliably and accurately. Results show how these methods may be used to accurately quantify cellular degradation machinery under various conditions, such as treatment with the endoplasmic reticulum stressor thapsigargin or ablation of the dynamin-related protein 1.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell?\n",
      "Stop? (Y/N)N\n",
      "36913670 Title:  Toward Understanding the Subcellular Distributions of Cholesterol and Sphingolipids Using High-Resolution NanoSIMS Imaging\n",
      "Abstract:  [\"Characterizing the subcellular distributions of biomolecules of interest is a basic inquiry that helps inform on the potential roles of these molecules in biological functions. Presently, the functions of specific lipid species and cholesterol are not well understood, partially because cholesterol and lipid species of interest are difficult to image with high spatial resolution but without perturbing them. Because cholesterol and lipids are relatively small and their distributions are influenced by noncovalent interactions with other biomolecules, functionalizing them with relatively large labels that permit their detection may alter their distributions in membranes and between organelles. This challenge has been surmounted by exploiting rare stable isotopes as labels that may be metabolically incorporated into cholesterol and lipids without altering their chemical compositions, and the Cameca NanoSIMS 50 instrument's ability to image rare stable isotope labels with high spatial resolution. This Account covers the use of secondary ion mass spectrometry (SIMS) performed with a Cameca NanoSIMS 50 instrument for imaging cholesterol and sphingolipids in the membranes of mammalian cells. The NanoSIMS 50 detects monatomic and diatomic secondary ions ejected from the sample to map the elemental and isotopic composition at the surface of the sample with better than 50 nm lateral resolution and 5 nm depth resolution. Much effort has focused on using NanoSIMS imaging of rare isotope-labeled cholesterol and sphingolipids for testing the long-standing hypothesis that cholesterol and sphingolipids colocalize within distinct domains in the plasma membrane. By using a NanoSIMS 50 to image rare isotope-labeled cholesterol and sphingolipids in parallel with affinity-labeled proteins of interest, a hypothesis regarding the colocalization of specific membrane proteins with cholesterol and sphingolipids in distinct plasma membrane domains has been tested. NanoSIMS performed in a depth profiling mode has enabled imaging the intracellular distributions of cholesterol and sphingolipids. Important progress has also been made in developing a computational depth correction strategy for constructing more accurate three-dimensional (3D) NanoSIMS depth profiling images of intracellular component distribution without requiring additional measurements with complementary techniques or signal collection. This Account provides an overview of this exciting progress, focusing on the studies from our laboratory that shifted understanding of plasma membrane organization, and the development of enabling tools for visualizing intracellular lipids.\"]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell? multi-organelle?\n",
      "Stop? (Y/N)N\n",
      "36912880 Title:  Activity regulates a cell type-specific mitochondrial phenotype in zebrafish lateral line hair cells\n",
      "Abstract:  ['Hair cells of the inner ear are particularly sensitive to changes in mitochondria, the subcellular organelles necessary for energy production in all eukaryotic cells. There are over 30 mitochondrial deafness genes, and mitochondria are implicated in hair cell death following noise exposure, aminoglycoside antibiotic exposure, as well as in age-related hearing loss. However, little is known about the basic aspects of hair cell mitochondrial biology. Using hair cells from the zebrafish lateral line as a model and serial block-face scanning electron microscopy, we have quantifiably characterized a unique hair cell mitochondrial phenotype that includes (1) a high mitochondrial volume and (2) specific mitochondrial architecture: multiple small mitochondria apically, and a reticular mitochondrial network basally. This phenotype develops gradually over the lifetime of the hair cell. Disrupting this mitochondrial phenotype with a mutation in <i>opa1</i> impacts mitochondrial health and function. While hair cell activity is not required for the high mitochondrial volume, it shapes the mitochondrial architecture, with mechanotransduction necessary for all patterning, and synaptic transmission necessary for the development of mitochondrial networks. These results demonstrate the high degree to which hair cells regulate their mitochondria for optimal physiology and provide new insights into mitochondrial deafness.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: multi-organelle? only mitos discussed but maybe nucleus included\n",
      "Stop? (Y/N)N\n",
      "36921538 Title:  Multimodal study of CHI3L1 inhibition and its effect on angiogenesis, migration, immune response and refractive index of cellular structures in glioblastoma\n",
      "Abstract:  ['Glioblastoma is one of the most aggressive tumours with a poor response to treatment and a poor prognosis for patients. One of the proteins expressed in glioblastoma tissue is CHI3L1 (YKL-40), which is upregulated and known for its angiogenesis-supporting and pro-tumour immunomodulatory effects in a variety of cancers. In this paper we present the anti-angiogenic, anti-migratory and immunomodulatory effects of the compound G721-0282, an inhibitor of CHI3L1. The inhibitor-induced changes were investigated using conventional techniques as well as the novel label-free digital holographic tomography (DHT), a quantitative phase imaging technique that allows the reconstruction of the refractive index (RI), which is used as an image contrast for 3D visualisation of living cells. DHT allowed digital staining of individual cells and intercellular structures based only on their specific RI. Quantitative spatially resolved analysis of the RI data shows that the concentration of G721-0282 leads to significant changes in the density of cells and their intracellular structures (in particular the cytoplasm and nucleus), in the volume of lipid droplets and in protein concentrations. Studies in the U-87 MG glioblastoma cell line, THP-1 monocytes differentiated into macrophages, human microvascular endothelial cells (HMEC-1) and in the spheroid model of glioblastoma composed of U-87 MG, HMEC-1 and macrophages suggest that inhibition of CHI3L1 may have potential in the antitumour treatment of glioblastoma. In this paper, we also propose a spheroid model for in vitro studies that mimics this type of tumour.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell? multi-organelle?\n",
      "Stop? (Y/N)N\n",
      "36950762 Title:  Non-Invasive and Minute-Frequency 3D Tomographic Imaging Enabling Long-Term Spatiotemporal Observation of Single Cell Fate\n",
      "Abstract:  ['Non-invasive and rapid imaging technique at subcellular resolution is significantly important for multiple biological applications such as cell fate study. Label-free refractive-index (RI)-based 3D tomographic imaging constitutes an excellent candidate for 3D imaging of cellular structures, but its full potential in long-term spatiotemporal cell fate observation is locked due to the lack of an efficient integrated system. Here, a long-term 3D RI imaging system incorporating a cutting-edge white light diffraction phase microscopy module with spatiotemporal stability, and an acoustofluidic device to roll and culture single cells in a customized live cell culture chamber is reported. Using this system, 3D RI imaging experiments are conducted for 250 cells and demonstrate efficient cell identification with high accuracy. Importantly, long-term and frequency-on-demand 3D RI imaging of K562 and MCF-7 cancer cells reveal different characteristics during normal cell growth, drug-induced cell apoptosis, and necrosis of drug-treated cells. Overall, it is believed that the proposed 3D tomographic imaging technique opens up a new avenue for visualizing intracellular structures and will find many applications such as disease diagnosis and nanomedicine.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: probably not, but language vague. \"intracellular structures\"\n",
      "Stop? (Y/N)N\n",
      "37036984 Title:  A spatially resolved elemental nanodomain organization within acidocalcisomes in Trypanosoma cruzi\n",
      "Abstract:  ['How are ions distributed in the three-dimensional (3D) volume confined in a nanoscale compartment? Regulation of ionic flow in the intracellular milieu has been explained by different theoretical models and experimentally demonstrated for several compartments with microscale dimensions. Most of these models predict a homogeneous distribution of ions seconds or milliseconds after an initial diffusion step formed at the ion translocation site, leaving open questions when it comes to ion/element distribution in spaces/compartments with nanoscale dimensions. Due to the influence of compartment size on the regulation of ionic flow, theoretical variations of classical models have been proposed, suggesting heterogeneous distributions of ions/elements within nanoscale compartments. Nonetheless, such assumptions have not been fully proven for the 3D volume of an organelle. In this work, we used a combination of cutting-edge electron microscopy techniques to map the 3D distribution of diffusible elements within the whole volume of acidocalcisomes in trypanosomes. Cryofixed cells were analyzed by scanning transmission electron microscopy tomography combined with elemental mapping using a high-performance setup of X-ray detectors. Results showed the existence of elemental nanodomains within the acidocalcisomes, where cationic elements display a self-excluding pattern. These were validated by Pearson correlation analysis and in silico molecular dynamic simulations. Formation of element domains within the 3D space of an organelle is demonstrated. Distribution patterns that support the electrodiffusion theory proposed for nanophysiology models have been found. The experimental pipeline shown here can be applied to a variety of models where ion mobilization plays a crucial role in physiological processes.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: single-organelle scale\n",
      "Stop? (Y/N)N\n",
      "37071854 Title:  Three-Dimensional Structure of Inner Ear Hair Cell Ribbon Synapses in a Zebrafish Model of Usher Syndrome Type 1B\n",
      "Abstract:  ['Our understanding of inner ear hair cell ultrastructure has heretofore relied upon two-dimensional imaging; however, serial block-face scanning electron microscopy (SBFSEM) changes this paradigm allowing for three-dimensional evaluation. We compared inner ear hair cells of the apical cristae in <i>myo7aa<sup>-/-</sup></i> null zebrafish, a model of human Usher Syndrome type 1B, to hair cells in wild-type zebrafish by SBFSEM to investigate possible ribbon synapse ultrastructural differences. Previously, it has been shown that compared to wild type, <i>myo7aa<sup>-/-</sup></i> zebrafish neuromast hair cells have fewer ribbon synapses yet similar ribbon areas. We expect the recapitulation of these results within the inner ear apical crista hair cells furthering the knowledge of three-dimensional ribbon synapse structure while resolving the feasibility of therapeutically targeting <i>myo7aa<sup>-/-</sup></i> mutant ribbons. In this report, we evaluated ribbon synapse number, volume, surface area, and sphericity. Localization of ribbons and their distance from the nearest innervation were also evaluated. We determined that <i>myo7aa<sup>-/-</sup></i> mutant ribbon synapses are smaller in volume and surface area; however, all other measurements were not significantly different from wild-type zebrafish. Because the ribbon synapses are nearly indistinguishable between the <i>myo7aa<sup>-/-</sup></i> mutant and wild type, it suggests that the ribbons are structurally receptive, supporting that therapeutic intervention may be feasible.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: organelles?\n",
      "Stop? (Y/N)N\n",
      "37090235 Title:  Surface-guided computing to analyze subcellular morphology and membrane-associated signals in 3D\n",
      "Abstract:  ['Signal transduction and cell function are governed by the spatiotemporal organization of membrane-associated molecules. Despite significant advances in visualizing molecular distributions by 3D light microscopy, cell biologists still have limited quantitative understanding of the processes implicated in the regulation of molecular signals at the whole cell scale. In particular, complex and transient cell surface morphologies challenge the complete sampling of cell geometry, membrane-associated molecular concentration and activity and the computing of meaningful parameters such as the cofluctuation between morphology and signals. Here, we introduce u-Unwrap3D, a framework to remap arbitrarily complex 3D cell surfaces and membrane-associated signals into equivalent lower dimensional representations. The mappings are bidirectional, allowing the application of image processing operations in the data representation best suited for the task and to subsequently present the results in any of the other representations, including the original 3D cell surface. Leveraging this surface-guided computing paradigm, we track segmented surface motifs in 2D to quantify the recruitment of Septin polymers by blebbing events; we quantify actin enrichment in peripheral ruffles; and we measure the speed of ruffle movement along topographically complex cell surfaces. Thus, u-Unwrap3D provides access to spatiotemporal analyses of cell biological parameters on unconstrained 3D surface geometries and signals.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: focus on cell surface, no intracellular structures\n",
      "Stop? (Y/N)N\n",
      "37131779 Title:  Surface-guided computing to analyze subcellular morphology and membrane-associated signals in 3D\n",
      "Abstract:  ['Signal transduction and cell function are governed by the spatiotemporal organization of membrane-associated molecules. Despite significant advances in visualizing molecular distributions by 3D light microscopy, cell biologists still have limited quantitative understanding of the processes implicated in the regulation of molecular signals at the whole cell scale. In particular, complex and transient cell surface morphologies challenge the complete sampling of cell geometry, membrane-associated molecular concentration and activity and the computing of meaningful parameters such as the cofluctuation between morphology and signals. Here, we introduce u-Unwrap3D, a framework to remap arbitrarily complex 3D cell surfaces and membrane-associated signals into equivalent lower dimensional representations. The mappings are bidirectional, allowing the application of image processing operations in the data representation best suited for the task and to subsequently present the results in any of the other representations, including the original 3D cell surface. Leveraging this surface-guided computing paradigm, we track segmented surface motifs in 2D to quantify the recruitment of Septin polymers by blebbing events; we quantify actin enrichment in peripheral ruffles; and we measure the speed of ruffle movement along topographically complex cell surfaces. Thus, u-Unwrap3D provides access to spatiotemporal analyses of cell biological parameters on unconstrained 3D surface geometries and signals.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: duplicate of above\n",
      "Stop? (Y/N)N\n",
      "37156644 Title:  Direct Visualization and Restoration of Metallic Ion-Induced Subcellular Ultrastructural Remodeling\n",
      "Abstract:  [\"Analysis of cellular ultrastructure dynamics and metal ions' fate can provide insights into the interaction between living organisms and metal ions. Here, we directly visualize the distribution of biogenic metallic aggregates, ion-induced subcellular reorganization, and the corresponding regulation effect in yeast by the near-native 3D imaging approach, cryo-soft X-ray tomography (cryo-SXT). By comparative 3D morphometric assessment, we observe the gold ions disrupting cellular organelle homeostasis, resulting in noticeable distortion and folding of vacuoles, apparent fragmentation of mitochondria, extreme swelling of lipid droplets, and formation of vesicles. The reconstructed 3D architecture of treated yeast demonstrates ∼65% of Au-rich sites in the periplasm, a comprehensive quantitative assessment unobtained by TEM. We also observe some AuNPs in rarely identified subcellular sites, namely, mitochondria and vesicles. Interestingly, the amount of gold deposition is positively correlated with the volume of lipid droplets. Shifting the external starting pH to near-neutral results in the reversion of changes in organelle architectures, boosting the amount of biogenic Au nanoparticles, and increasing cell viability. This study provides a strategy to analyze the metal ions-living organism interaction from subcellular architecture and spatial localization perspectives.\"]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments:  \n",
      "Stop? (Y/N)N\n",
      "37157259 Title:  Single-shot refractive index slice imaging using spectrally multiplexed optical transfer function reshaping\n",
      "Abstract:  ['The refractive index (RI) of cells and tissues is crucial in pathophysiology as a noninvasive and quantitative imaging contrast. Although its measurements have been demonstrated using three-dimensional quantitative phase imaging methods, these methods often require bulky interferometric setups or multiple measurements, which limits the measurement sensitivity and speed. Here, we present a single-shot RI imaging method that visualizes the RI of the in-focus region of a sample. By exploiting spectral multiplexing and optical transfer function engineering, three color-coded intensity images of a sample with three optimized illuminations were simultaneously obtained in a single-shot measurement. The measured intensity images were then deconvoluted to obtain the RI image of the in-focus slice of the sample. As a proof of concept, a setup was built using Fresnel lenses and a liquid-crystal display. For validation purposes, we measured microspheres of known RI and cross-validated the results with simulated results. Various static and highly dynamic biological cells were imaged to demonstrate that the proposed method can conduct single-shot RI slice imaging of biological samples with subcellular resolution.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell? multi-organelle? vague language\n",
      "Stop? (Y/N)N\n",
      "37167977 Title:  Human lung virtual histology by multi-scale x-ray phase-contrast computed tomography\n",
      "Abstract:  ['<i>Objectives.</i>As the central organ of the respiratory system, the human lung is responsible for supplying oxygen to the blood, which reaches the erythrocytes by diffusion through the alveolar walls and is then distributed throughout the body. By exploiting the difference in electron density detected by a phase shift in soft tissue, high-resolution x-ray phase-contrast computed tomography (XPCT) can resolve biological structures in a sub-<i>μ</i>m range, shedding new light on the three-dimensional structure of the lungs, physiological functions and pathological mechanisms.<i>Approach.</i>This work presents both synchrotron and laboratory XPCT results of postmortem tissue from autopsies and biopsies embedded with various preparation protocols such as precision-cut lung slices, cryogenically fixed lung tissue, as well as paraffin and alcohol fixed tissue. The selection of pathological abnormalities includes channel of Lambert, bronchus-associated lymphoid tissue and alveolar capillary dysplasia with misalignment of pulmonary veins. Subsequently, quantification and visualization approaches are presented.<i>Main results.</i>The overall high image quality even of in-house XPCT scans for the case of FFPE biopsies can be exploited for a wide range of pulmonary pathologies and translated to dedicated and optimized instrumentation which could be operated in clinical setting. By using synchrotron radiation, contrast can be further increased to resolve sub-<i>μ</i>m sized features down to the sub-cellular level. The results demonstrate that a wide range of preparation protocols including sample mounting in liquids can be used.<i>Significance.</i>With XPCT, poorly understood 3D structures can be identified in larger volume overview and subsequently studied in more detail at higher resolution. With the full 3D structure, the respective physiological functions of airways or vascular networks, and the different pathophysiologic mechanisms can be elucidated or at least underpinned with structural data. Moreover, synchrotron data can be used to validate laboratory protocols and provide ground truth for standardizing the method.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: tissue scale\n",
      "Stop? (Y/N)N\n",
      "37169939 Title:  Vacuolar control of stomatal opening revealed by 3D imaging of the guard cells\n",
      "Abstract:  ['Land plants regulate their photosynthesis and water transpiration by exchanging gases (CO<sub>2</sub> and H<sub>2</sub>O<sub>vapour</sub>) with the atmosphere. These exchanges take place through microscopic valves, called stomata, on the leaf surface. The opening of the stomata is regulated by two guard cells that actively and reversibly modify their turgor pressure to modulate the opening of the stomatal pores. Stomatal function depends on the regulation of the ion transport capacities of cell membranes as well as on the modification of the subcellular organisation of guard cells. Here we report how the vacuolar and cytosolic compartments of guard cells quantitatively participate in stomatal opening. We used a genetically encoded biosensor to visualise changes in ionic concentration during stomatal opening. The 3D reconstruction of living guard cells shows that the vacuole is the responsible for the change in guard cell volume required for stomatal opening.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: multi-organelle?\n",
      "Stop? (Y/N)N\n",
      "37184339 Title:  Nanometer Resolution Mass Spectro-Microtomography for In-Depth Anatomical Profiling of Single Cells\n",
      "Abstract:  ['Visually identifying the molecular changes in single cells is of great importance for unraveling fundamental cellular functions as well as disease mechanisms. Herein, we demonstrated a mass spectro-microtomography with an optimal voxel resolution of ∼300 × 300 × 25 nm<sup>3</sup>, which enables three-dimensional tomography of chemical substances in single cells. This mass imaging method allows for the distinguishment of abundant endogenous and exogenous molecules in subcellular structures. Combined with statistical analysis, we demonstrated this method for spatial metabolomics analysis of drug distribution and subsequent molecular damages caused by intracellular drug action. More interestingly, thanks to the nanoprecision ablation depth (∼12 nm), we realized metabolomics profiling of cell membrane without the interference of cytoplasm and improved the distinction of cancer cells from normal cells. Our current method holds great potential to be a powerful tool for spatially resolved single-cell metabolomics analysis of chemical components during complex biological processes.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: organelles?\n",
      "Stop? (Y/N)N\n",
      "37190020 Title:  Partial Inhibition of Complex I Restores Mitochondrial Morphology and Mitochondria-ER Communication in Hippocampus of APP/PS1 Mice\n",
      "Abstract:  [\"Alzheimer's disease (AD) has no cure. Earlier, we showed that partial inhibition of mitochondrial complex I (MCI) with the small molecule CP2 induces an adaptive stress response, activating multiple neuroprotective mechanisms. Chronic treatment reduced inflammation, Aβ and pTau accumulation, improved synaptic and mitochondrial functions, and blocked neurodegeneration in symptomatic APP/PS1 mice, a translational model of AD. Here, using serial block-face scanning electron microscopy (SBFSEM) and three-dimensional (3D) EM reconstructions combined with Western blot analysis and next-generation RNA sequencing, we demonstrate that CP2 treatment also restores mitochondrial morphology and mitochondria-endoplasmic reticulum (ER) communication, reducing ER and unfolded protein response (UPR) stress in the APP/PS1 mouse brain. Using 3D EM volume reconstructions, we show that in the hippocampus of APP/PS1 mice, dendritic mitochondria primarily exist as mitochondria-on-a-string (MOAS). Compared to other morphological phenotypes, MOAS have extensive interaction with the ER membranes, forming multiple mitochondria-ER contact sites (MERCS) known to facilitate abnormal lipid and calcium homeostasis, accumulation of Aβ and pTau, abnormal mitochondrial dynamics, and apoptosis. CP2 treatment reduced MOAS formation, consistent with improved energy homeostasis in the brain, with concomitant reductions in MERCS, ER/UPR stress, and improved lipid homeostasis. These data provide novel information on the MOAS-ER interaction in AD and additional support for the further development of partial MCI inhibitors as a disease-modifying strategy for AD.\"]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell?\n",
      "Stop? (Y/N)N\n",
      "37245582 Title:  The ultrastructural function of MLN64 in the late endosome-mitochondria membrane contact sites in placental cells\n",
      "Abstract:  ['The close apposition between two different organelles is critical in essential processes such as ion homeostasis, signaling, and lipid transition. However, information related to the structural features of membrane contact sites (MCSs) is limited. This study used immuno-electron microscopy and immuno-electron tomography (I-ET) to analyze the two- and three-dimensional structures of the late endosome-mitochondria contact sites in placental cells. Filamentous structures or tethers were identified that connected the late endosomes and mitochondria. Lamp1 antibody-labeled I-ET revealed enrichment of tethers in the MCSs. The cholesterol-binding endosomal protein metastatic lymph node 64 (MLN64) encoded by STARD3 was required for the formation of this apposition. The distance of the late endosome-mitochondria contact sites was <20\\xa0nm, shorter than that in STARD3-knockdown cells (<150\\xa0nm). The perturbation of cholesterol egress from the endosomes induced by U18666A treatment produced a longer distance in the contact sites than that in knockdown cells. The late endosome-mitochondria tethers failed to form correctly in STARD3-knockdown cells. Our results unravel the role of MLN64 involved in MCSs between late endosomes and mitochondria in placental cells.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: sounds like small regions at MCS, not whole-cell\n",
      "Stop? (Y/N)N\n",
      "37268154 Title:  Correlating cryo-super resolution radial fluctuations and dual-axis cryo-scanning transmission electron tomography to bridge the light-electron resolution gap\n",
      "Abstract:  ['Visualization of organelles and their interactions with other features in the native cell remains a challenge in modern biology. We have introduced cryo-scanning transmission electron tomography (CSTET), which can access 3D volumes on the scale of 1 micron with a resolution of nanometers, making it ideal for this task. Here we introduce two relevant advances: (a) we demonstrate the utility of multi-color super-resolution radial fluctuation light microscopy under cryogenic conditions (cryo-SRRF), and (b) we extend the use of deconvolution processing for dual-axis CSTET data. We show that cryo-SRRF nanoscopy is able to reach resolutions in the range of 100\\xa0nm, using commonly available fluorophores and a conventional widefield microscope for cryo-correlative light-electron microscopy. Such resolution aids in precisely identifying regions of interest before tomographic acquisition and enhances precision in localizing features of interest within the 3D reconstruction. Dual-axis CSTET tilt series data and application of entropy regularized deconvolution during post-processing results in close-to-isotropic resolution in the reconstruction without averaging. The integration of cryo-SRRF with deconvolved dual-axis CSTET provides a versatile workflow for studying unique objects in a cell.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell?\n",
      "Stop? (Y/N)N\n",
      "37352866 Title:  Real-time correction of chromatic aberration in optical fluorescence microscopy\n",
      "Abstract:  ['Multi-color fluorescence imaging is a powerful tool for studying the spatial relationships and interactions among sub-cellular structures in biological specimens. However, if improperly corrected, geometrical distortions caused by mechanical drift, refractive index mismatch, or chromatic aberration can lead to lower image resolution. In this paper, we present an extension of the image processing framework of Scipion by integrating a protocol called OFM Corrector, which corrects geometrical distortions in real-time using a B-spline-based elastic continuous registration technique. Our proposal provides a simple strategy to overcome chromatic aberration by digitally re-aligning color channels in multi-color fluorescence microscopy images, even in 3D or time. Our method relies on a geometrical calibration, which we do with fluorescent beads excited by different wavelengths of light and subsequently registered to get the elastic warp as a reference to correct chromatic shift. Our software is freely available with a user-friendly GUI and can be broadly used for various biological imaging problems. The paper presents a valuable tool for researchers working in light microscopy facilities.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: C\n",
      "Stop? (Y/N)sounds very technical, whole cell unlikely, but interactions between subcell structures mentioned\n",
      "37427938 Title:  Visualization of Organelles In Situ by Cryo-STEM Tomography\n",
      "Abstract:  ['Cryogenic electron microscopy (cryo-EM) relies on the imaging of biological or organic specimens embedded in their native aqueous medium; water is solidified into a glass (i.e., vitrified) without crystallization. The cryo-EM method is widely used to determine the structure of biological macromolecules recently at a near-atomic resolution. The approach has been extended to the study of organelles and cells using tomography, but the conventional mode of wide-field transmission EM imaging suffers a severe limitation in the specimen thickness. This has led to a practice of milling thin lamellae using a focused ion beam; the high resolution is obtained by subtomogram averaging from the reconstructions, but three-dimensional relations outside the remaining layer are lost. The thickness limitation can be circumvented by scanned probe imaging, similar to the scanning EM or the confocal laser scanning microscope. While scanning transmission electron microscopy (STEM) in materials science provides atomic resolution in single images, the sensitivity of cryogenic biological specimens to electron irradiation requires special considerations. This protocol presents a setup for cryo-tomography using STEM. The basic topical configuration of the microscope is described for both two- and three-condenser systems, while automation is provided by the non-commercial SerialEM software. Enhancements for batch acquisition and correlative alignment to previously-acquired fluorescence maps are also described. As an example, we show the reconstruction of a mitochondrion, pointing out the inner and outer membrane and calcium phosphate granules, as well as surrounding microtubules, actin filaments, and ribosomes. Cryo-STEM tomography excels in revealing the theater of organelles in the cytoplasm and, in some cases, even the nuclear periphery of adherent cells in culture.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: not whole-cell\n",
      "Stop? (Y/N)N\n",
      "37425741 Title:  3D reconstructions of parasite development and the intracellular niche of the microsporidian pathogen E. intestinalis\n",
      "Abstract:  ['Microsporidia are an early-diverging group of fungal pathogens that infect a wide range of hosts. Several microsporidian species infect humans, and infections can lead to fatal disease in immunocompromised individuals. As obligate intracellular parasites with highly reduced genomes, microsporidia are dependent on metabolites from their hosts for successful replication and development. Our knowledge of how microsporidian parasites develop inside the host remains rudimentary, and our understanding of the intracellular niche occupied by microsporidia has thus far relied largely on 2D TEM images and light microscopy. Here, we use serial block face scanning electron microscopy (SBF-SEM) to capture 3D snapshots of the human-infecting microsporidian, <i>Encephalitozoon intestinalis</i> , within host cells. We track the development of <i>E. intestinalis</i> through its life cycle, which allows us to propose a model for how its infection organelle, the polar tube, is assembled <i>de novo</i> in each developing spore. 3D reconstructions of parasite-infected cells provide insights into the physical interactions between host cell organelles and parasitophorous vacuoles, which contain the developing parasites. The host cell mitochondrial network is substantially remodeled during <i>E. intestinalis</i> infection, leading to mitochondrial fragmentation. SBF-SEM analysis shows changes in mitochondrial morphology in infected cells, and live-cell imaging provides insights into mitochondrial dynamics during infection. Together, our data provide insights into parasite development, polar tube assembly, and microsporidia-induced mitochondrial remodeling in the host cell.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments: N\n",
      "Stop? (Y/N)N\n",
      "37433616 Title:  Dehydration as alternative sample preparation for soft X-ray tomography\n",
      "Abstract:  [\"Soft X-ray tomography (SXT) is an imaging technique to visualise whole cells without fixation, staining, and sectioning. For SXT imaging, cells are cryopreserved and imaged at cryogenic conditions. Such 'near-to-native' state imaging is in high demand and initiated the development of the laboratory table-top SXT microscope. As many laboratories do not have access to cryogenic equipment, we asked ourselves whether SXT imaging is feasible on dry specimens. This paper shows how the dehydration of cells can be used as an alternative sample preparation to obtain ultrastructure information. We compare different dehydration processes on mouse embryonic fibroblasts in terms of ultrastructural preservation and shrinkage. Based on this analysis, we chose critical point (CPD) dried cells for SXT imaging. In comparison to cryopreserved and air-dried cells, CPD dehydrated cells show high structural integrity although with about 3-7 times higher X-ray absorption for cellular organelles. As the difference in X-ray absorption values between organelles is preserved, 3D anatomy of CPD-dried cells can be segmented and analysed, demonstrating the applicability of CPD-dried sample preparation for SXT imaging. LAY DESCRIPTION: Soft X-ray tomography (SXT) is an imaging technique that allows to see the internal structures of cells without the need for special treatments like fixation or staining. Typically, SXT imaging involves freezing and imaging cells at very low temperatures. However, since many labs lack the necessary equipment, we explored whether SXT imaging could be done on dry samples instead. We compared different dehydration methods and found that critical point drying (CPD) was the most promising for SXT imaging. CPD-dried cells showed high structural integrity, although they absorbed more X-rays than hydrated cells, demonstrating that CPD-dried sample preparation is a viable alternative for SXT imaging.\"]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell?\n",
      "Stop? (Y/N)N\n",
      "37449034 Title:  3D Ultrastructural Visualization of Mitosis Fidelity in Human Cells Using Serial Block Face Scanning Electron Microscopy (SBF-SEM)\n",
      "Abstract:  ['Errors in chromosome segregation during mitosis lead to chromosome instability, resulting in an unbalanced number of chromosomes in the daughter cells. Light microscopy has been used extensively to study chromosome missegregation by visualizing errors of the mitotic spindle. However, less attention has been paid to understanding spindle function in the broader context of intracellular structures and organelles during mitosis. Here, we outline a protocol to visualize chromosomes and endomembranes in mitosis, combining light microscopy and 3D volume electron microscopy, serial block-face scanning electron microscopy (SBF-SEM). SBF-SEM provides high-resolution imaging of large volumes and subcellular structures, followed by image analysis and 3D reconstruction. This protocol allows scientists to visualize the whole subcellular context of the spindle during mitosis.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: multi-organelle?\n",
      "Stop? (Y/N)N\n",
      "37455654 Title:  Targeted volume correlative light and electron microscopy of an environmental marine microorganism\n",
      "Abstract:  ['Photosynthetic microalgae are responsible for an important fraction of CO2 fixation and O2 production on Earth. Three-dimensional (3D) ultrastructural characterization of these organisms in their natural environment can contribute to a deeper understanding of their cell biology. However, the low throughput of volume electron microscopy (vEM) methods along with the complexity and heterogeneity of environmental samples pose great technical challenges. In the present study, we used a workflow based on a specific electron microscopy sample preparation method compatible with both light and vEM imaging in order to target one cell among a complex natural community. This method revealed the 3D subcellular landscape of a photosynthetic dinoflagellate, which we identified as Ensiculifera tyrrhenica, with quantitative characterization of multiple organelles. We show that this cell contains a single convoluted chloroplast and show the arrangement of the flagellar apparatus with its associated photosensitive elements. Moreover, we observed partial chromatin unfolding, potentially associated with transcription activity in these organisms, in which chromosomes are permanently condensed. Together with providing insights in dinoflagellate biology, this proof-of-principle study illustrates an efficient tool for the targeted ultrastructural analysis of environmental microorganisms in heterogeneous mixes.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments:  \n",
      "Stop? (Y/N)N\n",
      "37523497 Title:  Influence of Chemical and Genetic Manipulations on Cellular Organelles Quantified by Label-Free Optical Diffraction Tomography\n",
      "Abstract:  ['Label-free optical diffraction tomography provides three-dimensional imaging of cells and organelles, along with their refractive index (RI) and volume. These physical parameters are valuable for quantitative and accurate analysis of the subcellular microenvironment and its connections to intracellular biological properties. In biological and biochemical cell analysis, various invasive cell manipulations are used, such as temperature change, chemical fixation, live cell staining with fluorescent dye, and gene overexpression of exogenous proteins. However, it is not fully understood how these various manipulations affect the physicochemical properties of different organelles. In this study, we investigated the impact of these manipulations on the cellular properties of single HeLa cells. We found that after cell fixation and an increase in temperature, the RI value of organelles, such as the nucleus and cytoplasm, significantly decreased overall. Interestingly, unlike the cell nuclei, cytoplasmic RI values were hardly detected after membrane permeation, indicating that only intracytoplasmic components were largely lost. Additionally, our findings revealed that the expression of GFP and GFP-tagged proteins significantly increased the RI values of organelles in living cells compared to the less effective RI changes observed with chemical fluorescence staining for cell organelles. The result demonstrates that distinct types of invasive manipulations can alter the microenvironment of organelles in different ways. Our study sheds new light on how chemical and genetic manipulations affect organelles.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: 3D/whole cell?\n",
      "Stop? (Y/N)N\n",
      "37519903 Title:  Spatial topology of organelle is a new breast cancer cell classifier\n",
      "Abstract:  ['Genomics and proteomics have been central to identify tumor cell populations, but more accurate approaches to classify cell subtypes are still lacking. We propose a new methodology to accurately classify cancer cells based on their organelle spatial topology. Herein, we developed an organelle topology-based cell classification pipeline (OTCCP), which integrates artificial intelligence (AI) and imaging quantification to analyze organelle spatial distribution and inter-organelle topology. OTCCP was used to classify a panel of human breast cancer cells, grown as 2D monolayer or 3D tumor spheroids using early endosomes, mitochondria, and their inter-organelle contacts. Organelle topology allows for a highly precise differentiation between cell lines of different subtypes and aggressiveness. These findings lay the groundwork for using organelle topological profiling as a fast and efficient method for phenotyping breast cancer function as well as a discovery tool to advance our understanding of cancer cell biology at the subcellular level.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: 3D/whole cell?\n",
      "Stop? (Y/N)N\n",
      "37600951 Title:  Assessment of Toxoplasma gondii lytic cycle and the impact of a gene deletion using 3D label-free optical diffraction holotomography\n",
      "Abstract:  ['<i>Toxoplasma gondii</i> is a widespread single-celled intracellular eukaryotic apicomplexan protozoan parasite primarily associated with mammalian foetal impairment and miscarriage, including in humans. Is estimated that approximately one third of the human population worldwide is infected by this parasite. Here we used cutting-edge, label-free 3D quantitative optical diffraction holotomography to capture and evaluate the <i>Toxoplasma</i> lytic cycle (invasion, proliferation and egress) in real-time based on the refractive index distribution. In addition, we used this technology to analyse an engineered CRISPR-Cas9 <i>Toxoplasma</i> mutant to reveal differences in cellular physical properties when compared to the parental line. Collectively, these data support the use of holotomography as a powerful tool for the study of protozoan parasites and their interactions with their host cells.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: organelles? 3D whole cell?\n",
      "Stop? (Y/N)N\n",
      "37613159 Title:  Charting Cytoskeleton-Organelle Interplay in Living Cells Through High Resolution 3D Correlative Cryo-Imaging\n",
      "No abstract available, indicate Check Full Text (C)\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: no abstract available\n",
      "Stop? (Y/N)N\n",
      "37613169 Title:  The Role of Soft X-ray Tomography in Generating Whole-cell Models\n",
      "No abstract available, indicate Check Full Text (C)\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: no abstract available\n",
      "Stop? (Y/N)N\n",
      "37613241 Title:  From Organ to Organelle: towards a Multimodal 3D Cell Atlas of the Malaria Parasite Mosquito stages using Correlative Light, X-ray and volume Electron Microscopy\n",
      "No abstract available, indicate Check Full Text (C)\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: no abstract available\n",
      "Stop? (Y/N)N\n",
      "37613466 Title:  Soft X-ray Tomography for Mapping and Quantifying Intracellular Organelle Interactions\n",
      "No abstract available, indicate Check Full Text (C)\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: no abstract available\n",
      "Stop? (Y/N)N\n",
      "37613509 Title:  Soft X-ray Tomography Enables New Insights into the Coordinated Division of Organelle-like Symbiont in a Globally Distributed Unicellular Marine Haptophyte Alga\n",
      "No abstract available, indicate Check Full Text (C)\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: no abstract available\n",
      "Stop? (Y/N)N\n",
      "37613247 Title:  Charting Cytoskeleton-organelle Interplay in Living Cells Through High Resolution 3D Correlative Cryo-imaging\n",
      "No abstract available, indicate Check Full Text (C)\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: duplcate of 37613159 \n",
      "Stop? (Y/N)N\n",
      "37670547 Title:  Label-Free Intracellular Multi-Specificity in Yeast Cells by Phase-Contrast Tomographic Flow Cytometry\n",
      "Abstract:  [\"In-flow phase-contrast tomography provides a 3D refractive index of label-free cells in cytometry systems. Its major limitation, as with any quantitative phase imaging approach, is the lack of specificity compared to fluorescence microscopy, thus restraining its huge potentialities in single-cell analysis and diagnostics. Remarkable results in introducing specificity are obtained through artificial intelligence (AI), but only for adherent cells. However, accessing the 3D fluorescence ground truth and obtaining accurate voxel-level co-registration of image pairs for AI training is not viable for high-throughput cytometry. The recent statistical inference approach is a significant step forward for label-free specificity but remains limited to cells' nuclei. Here, a generalized computational strategy based on a self-consistent statistical inference to achieve intracellular multi-specificity is shown. Various subcellular compartments (i.e., nuclei, cytoplasmic vacuoles, the peri-vacuolar membrane area, cytoplasm, vacuole-nucleus contact site) can be identified and characterized quantitatively at different phases of the cells life cycle by using yeast cells as a biological model. Moreover, for the first time, virtual reality is introduced for handling the information content of multi-specificity in single cells. Full fruition is proofed for exploring and interacting with 3D quantitative biophysical parameters of the identified compartments on demand, thus opening the route to a metaverse for 3D microscopy.\"]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell?\n",
      "Stop? (Y/N)N\n",
      "37707263 Title:  Structured illumination phase and fluorescence microscopy for bioimaging\n",
      "Abstract:  ['This study presents a dual-modality microscopic imaging approach that combines quantitative phase microscopy and fluorescence microscopy based on structured illumination (SI) to provide structural and functional information for the same sample. As the first imaging modality, structured illumination digital holographic microscopy (SI-DHM) is implemented along the transmission beam path. SI-DHM acts as a label-free, noninvasive approach and provides high-contrast and quantitative phase images utilizing the refractive index contrast of the inner structures of samples against the background. As the second imaging modality, structured illumination (fluorescence) microscopy (SIM) is constructed along the reflection beam path. SIM utilizes fluorescent labeling and provides super-resolution images for specific functional structures of samples. We first experimentally demonstrated phase imaging of SI-DHM on rice leaves and fluorescence (SIM) imaging on mouse kidney sections. Then, we demonstrated dual-modality imaging of biological samples, using DHM to acquire the overall cell morphology and SIM to obtain specific functional structures. These results prove that the proposed technique is of great importance in biomedical studies, such as providing insight into cell physiology by visualizing and quantifying subcellular structures.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell/multiorganelle?\n",
      "Stop? (Y/N)N\n",
      "37737610 Title:  Ultrastructure and 3D reconstruction of a diplonemid protist (Diplonemea) and its novel membranous organelle\n",
      "Abstract:  ['The knowledge of cell biology of a eukaryotic group is essential for correct interpretation of ecological and molecular data. Although diplonemid protists are one of the most species-rich lineages of marine eukaryotes, only very fragmentary information is available about the cellular architecture of this taxonomically diverse group. Here, a large serial block-face scanning electron microscopy data set complemented with light and fluorescence microscopy allowed the first detailed three-dimensional reconstruction of a diplonemid species. We describe numerous previously unknown peculiarities of the cellular architecture and cell division characteristic for diplonemid flagellates, and illustrate the obtained results with multiple three-dimensional models, comprehensible for non-specialists in protist ultrastructure.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments:  \n",
      "Stop? (Y/N)N\n",
      "37749240 Title:  Plantorganelle Hunter is an effective deep-learning-based method for plant organelle phenotyping in electron microscopy\n",
      "Abstract:  ['Accurate delineation of plant cell organelles from electron microscope images is essential for understanding subcellular behaviour and function. Here we develop a deep-learning pipeline, called the organelle segmentation network (OrgSegNet), for pixel-wise segmentation to identify chloroplasts, mitochondria, nuclei and vacuoles. OrgSegNet was evaluated on a large manually annotated dataset collected from 19 plant species and achieved state-of-the-art segmentation performance. We defined three digital traits (shape complexity, electron density and cross-sectional area) to track the quantitative features of individual organelles in 2D images and released an open-source web tool called Plantorganelle Hunter for quantitatively profiling subcellular morphology. In addition, the automatic segmentation method was successfully applied to a serial-sectioning scanning microscope technique to create a 3D cell model that offers unique views of the morphology and distribution of these organelles. The functionalities of Plantorganelle Hunter can be easily operated, which will increase efficiency and productivity for the plant science community, and enhance understanding of subcellular biology.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?whole cell? real or integrated model/virtual?\n",
      "whoops! invalid answer\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell? real or integrated model/virtual?\n",
      "Stop? (Y/N)N\n",
      "37799690 Title:  Volumetric live-cell autofluorescence imaging using Fourier light-field microscopy\n",
      "Abstract:  ['This study introduces a rapid, volumetric live-cell imaging technique for visualizing autofluorescent sub-cellular structures and their dynamics by employing high-resolution Fourier light-field microscopy. We demonstrated this method by capturing lysosomal autofluorescence in fibroblasts and HeLa cells. Additionally, we conducted multicolor imaging to simultaneously observe lysosomal autofluorescence and fluorescently-labeled organelles such as lysosomes and mitochondria. We further analyzed the data to quantify the interactions between lysosomes and mitochondria. This research lays the foundation for future exploration of native cellular states and functions in three-dimensional environments, effectively reducing photodamage and eliminating the necessity for exogenous labels.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole-cell?\n",
      "Stop? (Y/N)N\n",
      "37805154 Title:  Beyond the surface: Investigation of tumorsphere morphology using volume electron microscopy\n",
      "Abstract:  ['The advent of volume electron microscopy (vEM) has provided unprecedented insights into cellular and subcellular organization, revolutionizing our understanding of cancer biology. This study presents a previously unexplored comparative analysis of the ultrastructural disparities between cancer cells cultured as monolayers and tumorspheres. By integrating a robust workflow that incorporates high-pressure freezing followed by freeze substitution (HPF/FS), serial block face scanning electron microscopy (SBF-SEM), manual and deep learning-based segmentation, and statistical analysis, we have successfully generated three-dimensional (3D) reconstructions of monolayer and tumorsphere cells, including their subcellular organelles. Our findings reveal a significant degree of variation in cellular morphology in tumorspheres. We observed the increased prevalence of nuclear envelope invaginations in tumorsphere cells compared to monolayers. Furthermore, we detected a diverse range of mitochondrial morphologies exclusively in tumorsphere cells, as well as intricate cellular interconnectivity within the tumorsphere architecture. These remarkable ultrastructural differences emphasize the use of tumorspheres as a superior model for cancer research due to their relevance to in vivo conditions. Our results strongly advocate for the utilization of tumorsphere cells in cancer research studies, enhancing the precision and relevance of experimental outcomes, and ultimately accelerating therapeutic advancements.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: single cells distinguishable in tissue context? whole cell? \n",
      "Stop? (Y/N)N\n",
      "37808751 Title:  Whole-cell multi-target single-molecule super-resolution imaging in 3D with microfluidics and a single-objective tilted light sheet\n",
      "Abstract:  ['Multi-target single-molecule super-resolution fluorescence microscopy offers a powerful means of understanding the distributions and interplay between multiple subcellular structures at the nanoscale. However, single-molecule super-resolution imaging of whole mammalian cells is often hampered by high fluorescence background and slow acquisition speeds, especially when imaging multiple targets in 3D. In this work, we have mitigated these issues by developing a steerable, dithered, single-objective tilted light sheet for optical sectioning to reduce fluorescence background and a pipeline for 3D nanoprinting microfluidic systems for reflection of the light sheet into the sample and for efficient and automated solution exchange. By combining these innovations with PSF engineering for nanoscale localization of individual molecules in 3D, deep learning for analysis of overlapping emitters, active 3D stabilization for drift correction and long-term imaging, and Exchange-PAINT for sequential multi-target imaging without chromatic offsets, we demonstrate whole-cell multi-target 3D single-molecule super-resolution imaging with improved precision and imaging speed.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: organelles? \n",
      "Stop? (Y/N)N\n",
      "37852350 Title:  3DCNAS: A universal method for predicting the location of fluorescent organelles in living cells in three-dimensional space\n",
      "Abstract:  ['Cellular biology research relies on microscopic imaging techniques for studying the complex structures and dynamic processes within cells. Fluorescence microscopy provides high sensitivity and subcellular resolution but has limitations such as photobleaching and sample preparation challenges. Transmission light microscopy offers a label-free alternative but lacks contrast for detailed interpretation. Deep learning methods have shown promise in analyzing cell images and extracting meaningful information. However, accurately learning and simulating diverse subcellular structures remain challenging. In this study, we propose a method named three-dimensional cell neural architecture search (3DCNAS) to predict subcellular structures of fluorescence using unlabeled transmitted light microscope images. By leveraging the automated search capability of differentiable neural architecture search (NAS), our method partially mitigates the issues of overfitting and underfitting caused by the distinct details of various subcellular structures. Furthermore, we apply our method to analyze cell dynamics in genome-edited human induced pluripotent stem cells during mitotic events. This allows us to study the functional roles of organelles and their involvement in cellular processes, contributing to a comprehensive understanding of cell biology and offering insights into disease pathogenesis.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole-cell image or generative model?\n",
      "Stop? (Y/N)N\n",
      "37856022 Title:  Airway Cells 3D Reconstruction via Manual and Machine-Learning Aided Segmentation of Volume EM Datasets\n",
      "Abstract:  ['Volume electron microscopy (vEM) is a high-resolution imaging technique capable of revealing the 3D structure of cells, tissues, and model organisms. This imaging modality is gaining prominence due to its ability to provide a comprehensive view of cells at the nanometer scale. The visualization and quantitative analysis of individual subcellular structures however\\xa0requires segmentation of each 2D electron micrograph slice of the 3D vEM dataset; this process is extremely laborious de facto limiting its applications and throughput. To address these limitations, deep learning approaches have been recently developed including Empanada-Napari plugin, an open-source tool for automated segmentation based on a Panoptic-DeepLab (PDL) architecture. In this chapter, we provide a step-by-step protocol describing the process of manual segmentation using 3dMOD within the IMOD package and the process of automated segmentation using Empanada-Napari plugins for the 3D reconstruction of airway cellular structures.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell/multi-organelle?\n",
      "Stop? (Y/N)N\n",
      "37885921 Title:  Neodymium acetate as a contrast agent for X-ray phase-contrast tomography\n",
      "Abstract:  [StringElement('X-ray phase-contrast tomography (XPCT) is a non-destructive, three-dimensional imaging modality that provides higher contrast in soft tissue than absorption-based CT and allows one to cover the cytoarchitecture from the centi- and millimeter scale down to the nanoscale. To further increase contrast and resolution of XPCT, for example, in view of addressing connectivity issues in the central nervous system (CNS), metal staining is indispensable. However, currently used protocols, for example, based on osmium and/or uranium are less suited for XPCT, due to an excessive <math xmlns=\"http://www.w3.org/1998/Math/MathML\"><mrow><mi>β</mi><mo>/</mo><mi>δ</mi></mrow></math>-ratio. In this work, we explore the suitability of different staining agents for XPCT. Particularly, neodymium(III)-acetate (NdAc), which has recently been proposed as a non-toxic, non-radioactive easy to use alternative contrast agent for uranyl acetate (UAc) in electron microscopy, is investigated. Due to its vertical proximity to UAc in the periodic table, similar chemical but better suited optical properties for phase contrast can be expected.', attributes={'Label': 'PURPOSE', 'NlmCategory': 'UNASSIGNED'}), StringElement('Differently stained whole eye samples of wild type mouse and tissues of the CNS are embedded into EPON epoxy resin and scanned using synchrotron as well as with laboratory radiation. Phase retrieval is performed on the projection images, followed by tomographic reconstruction, which enables a quantitative analysis based on the reconstructed electron densities. Segmentation techniques and rendering software is used to visualize structures of interest in the sample.', attributes={'Label': 'APPROACH', 'NlmCategory': 'UNASSIGNED'}), StringElement('We show that staining neuronal samples with NdAc enhances contrast, in particular for laboratory scans, allowing high-resolution imaging of biological soft tissue in-house. For the example of murine retina, specifically rods and cones as well as the sclera and the Ganglion cell layer seem to be targeted by the stain. A comparison of electron density by the evaluation of histograms allowed to determine quantitative measures to describe the difference between the examined stains.', attributes={'Label': 'RESULTS', 'NlmCategory': 'UNASSIGNED'}), StringElement('The results suggest NdAc to be an effective stain for XPCT, with a preferential binding to anionic groups, such as phosphate and carboxyl groups at cell surfaces, targeting certain layers of the retina with a stronger selectivity compared to other staining agents. Due to the advantageous X-ray optical properties, the stain seems particularly well-suited for phase contrast, with a comparably small number density and an overall superior image quality at laboratory sources.', attributes={'Label': 'CONCLUSION', 'NlmCategory': 'UNASSIGNED'})]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: tissue scale\n",
      "Stop? (Y/N)N\n",
      "37908116 Title:  Automated quantification of vacuole fusion and lipophagy in Saccharomyces cerevisiae from fluorescence and cryo-soft X-ray microscopy data using deep learning\n",
      "Abstract:  ['During starvation in the yeast <i>Saccharomyces cerevisiae</i> vacuolar vesicles fuse and lipid droplets (LDs) can become internalized into the vacuole in an autophagic process named lipophagy. There is a lack of tools to quantitatively assess starvation-induced vacuole fusion and lipophagy in intact cells with high resolution and throughput. Here, we combine soft X-ray tomography (SXT) with fluorescence microscopy and use a deep-learning computational approach to visualize and quantify these processes in yeast. We focus on yeast homologs of mammalian NPC1 (NPC intracellular cholesterol transporter 1; Ncr1 in yeast) and NPC2 proteins, whose dysfunction leads to Niemann Pick type C (NPC) disease in humans. We developed a convolutional neural network (CNN) model which classifies fully fused versus partially fused vacuoles based on fluorescence images of stained cells. This CNN, named Deep Yeast Fusion Network (DYFNet), revealed that cells lacking Ncr1 (<i>ncr1∆</i> cells) or Npc2 (<i>npc2∆</i> cells) have a reduced capacity for vacuole fusion. Using a second CNN model, we implemented a pipeline named LipoSeg to perform automated instance segmentation of LDs and vacuoles from high-resolution reconstructions of X-ray tomograms. From that, we obtained 3D renderings of LDs inside and outside of the vacuole in a fully automated manner and additionally measured droplet volume, number, and distribution. We find that <i>ncr1∆</i> and <i>npc2∆</i> cells could ingest LDs into vacuoles normally but showed compromised degradation of LDs and accumulation of lipid vesicles inside vacuoles. Our new method is versatile and allows for analysis of vacuole fusion, droplet size and lipophagy in intact cells.<b>Abbreviations:</b> BODIPY493/503: 4,4-difluoro-1,3,5,7,8-pentamethyl-4-bora-3a,4a-diaza-<i>s</i>-Indacene; BPS: bathophenanthrolinedisulfonic acid disodium salt hydrate; CNN: convolutional neural network; DHE; dehydroergosterol; <i>npc2∆</i>, yeast deficient in Npc2; DSC, Dice similarity coefficient; EM, electron microscopy; EVs, extracellular vesicles; FIB-SEM, focused ion beam milling-scanning electron microscopy; FM 4-64, <i>N</i>-(3-triethylammoniumpropyl)-4-(6-[4-{diethylamino} phenyl] hexatrienyl)-pyridinium dibromide; LDs, lipid droplets; Ncr1, yeast homolog of human NPC1 protein; <i>ncr1∆</i>, yeast deficient in Ncr1; NPC, Niemann Pick type C; NPC2, Niemann Pick type C homolog; OD<sub>600</sub>, optical density at 600 nm; ReLU, rectifier linear unit; PPV, positive predictive value; NPV, negative predictive value; MCC, Matthews correlation coefficient; SXT, soft X-ray tomography; UV, ultraviolet; YPD, yeast extract peptone dextrose.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments:  \n",
      "Stop? (Y/N)N\n",
      "37929966 Title:  Visualization and Quantification of Endogenous Intra-Organelle Protein Interactions at ER-Mitochondria Contact Sites by Proximity Ligation Assays\n",
      "Abstract:  ['Membrane contact sites (MCSs) are areas of close membrane proximity that allow and regulate the dynamic exchange of diverse biomolecules (i.e., calcium and lipids) between the juxtaposed organelles without involving membrane fusion. MCSs are essential for cellular homeostasis, and their functions are ensured by the resident components, which often exist as multimeric protein complexes. MCSs often involve the endoplasmic reticulum (ER), a major site of lipid synthesis and cellular calcium storage, and are particularly important for organelles, such as the mitochondria, which are excluded from the classical vesicular transport pathways. In the last years, MCSs between the ER and mitochondria have been extensively studied, as their functions strongly impact cellular metabolism/bioenergetics. Several proteins have started to be identified at these contact sites, including membrane tethers, calcium channels, and lipid transfer proteins, thus raising the need for new methodologies and technical approaches to study these MCS components. Here, we describe a protocol consisting of combined technical approaches, that include proximity ligation assay (PLA), mitochondria staining, and 3D imaging segmentation, that allows the detection of proteins that are physically close (>40 nm) to each other and that reside on the same membrane at ER-mitochondria MCSs. For instance, we used two ER-anchored lipid transfer proteins, ORP5 and ORP8, which have previously been shown to interact and localize at ER-mitochondria and ER-plasma membrane MCSs. By associating the ORP5-ORP8 PLA with cell imaging software analysis, it was possible to estimate the distance of the ORP5-ORP8 complex from the mitochondrial surface and determine that about 50% of ORP5-ORP8 PLA interaction occurs at ER subdomains in close proximity to mitochondria.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: not whole-cell multi-organelle\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Stop? (Y/N)N\n",
      "37933490 Title:  Deep learning, 3D ultrastructural analysis reveals quantitative differences in platelet and organelle packing in COVID-19/SARSCoV2 patient-derived platelets\n",
      "Abstract:  ['Platelets contribute to COVID-19 clinical manifestations, of which microclotting in the pulmonary vasculature has been a prominent symptom. To investigate the potential diagnostic contributions of overall platelet morphology and their α-granules and mitochondria to the understanding of platelet hyperactivation and micro-clotting, we undertook a 3D ultrastructural approach. Because differences might be small, we used the high-contrast, high-resolution technique of focused ion beam scanning EM (FIB-SEM) and employed deep learning computational methods to evaluate nearly 600 individual platelets and 30\\u2009000 included organelles within three healthy controls and three severely ill COVID-19 patients. Statistical analysis reveals that the α-granule/mitochondrion-to-plateletvolume ratio is significantly greater in COVID-19 patient platelets indicating a denser packing of organelles, and a more compact platelet. The COVID-19 patient platelets were significantly smaller -by 35% in volume - with most of the difference in organelle packing density being due to decreased platelet size. There was little to no 3D ultrastructural evidence for differential activation of the platelets from COVID-19 patients. Though limited by sample size, our studies suggest that factors outside of the platelets themselves are likely responsible for COVID-19 complications. Our studies show how deep learning 3D methodology can become the gold standard for 3D ultrastructural studies of platelets.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole-cell reconstructions? \n",
      "Stop? (Y/N)N\n",
      "37946316 Title:  Three-dimensional architecture of granulosa cell derived from oocyte cumulus complex, revealed by FIB-SEM\n",
      "Abstract:  ['The oocyte cumulus complex is mainly composed of an oocyte, the perivitelline space, zona pellucida and numerous granulosa cells. The cumulus granulosa cells (cGCs) provide a particularly important microenvironment for oocyte development, regulating its growth, maturation and meiosis. In this study, we studied the internal structures and cell-to-cell connections of mouse cGCs using focused ion beam scanning electron microscopy (FIB-SEM). We reconstructed three-dimensional models to display characteristic connections between the oocyte and cGCs, and to illustrate various main organelles in cGCs together with their interaction relationship. A special form of cilium identified in granulosa cell was never reported in previous literature.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell w/ membrane?\n",
      "Stop? (Y/N)N\n",
      "37948126 Title:  Membrane contact site detection (MCS-DETECT) reveals dual control of rough mitochondria-ER contacts\n",
      "Abstract:  ['Identification and morphological analysis of mitochondria-ER contacts (MERCs) by fluorescent microscopy is limited by subpixel resolution interorganelle distances. Here, the membrane contact site (MCS) detection algorithm, MCS-DETECT, reconstructs subpixel resolution MERCs from 3D super-resolution image volumes. MCS-DETECT shows that elongated ribosome-studded riboMERCs, present in HT-1080 but not COS-7 cells, are morphologically distinct from smaller smooth contacts and larger contacts induced by mitochondria-ER linker expression in COS-7 cells. RiboMERC formation is associated with increased mitochondrial potential, reduced in Gp78 knockout HT-1080 cells and induced by Gp78 ubiquitin ligase activity in COS-7 and HeLa cells. Knockdown of riboMERC tether RRBP1 eliminates riboMERCs in both wild-type and Gp78 knockout HT-1080 cells. By MCS-DETECT, Gp78-dependent riboMERCs present complex tubular shapes that intercalate between and contact multiple mitochondria. MCS-DETECT of 3D whole-cell super-resolution image volumes, therefore, identifies novel dual control of tubular riboMERCs, whose formation is dependent on RRBP1 and size modulated by Gp78 E3 ubiquitin ligase activity.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell 3D reconstruction?\n",
      "Stop? (Y/N)N\n",
      "37980360 Title:  Adaptive traits of cysts of the snow alga Sanguina nivaloides unveiled by 3D subcellular imaging\n",
      "Abstract:  ['Sanguina nivaloides is the main alga forming red snowfields in high mountains and Polar Regions. It is non-cultivable. Analysis of environmental samples by X-ray tomography, focused-ion-beam scanning-electron-microscopy, physicochemical and physiological characterization reveal adaptive traits accounting for algal capacity to reside in snow. Cysts populate liquid water at the periphery of ice, are photosynthetically active, can survive for months, and are sensitive to freezing. They harbor a wrinkled plasma membrane expanding the interface with environment. Ionomic analysis supports a cell efflux of K<sup>+</sup>, and assimilation of phosphorus. Glycerolipidomic analysis confirms a phosphate limitation. The chloroplast contains thylakoids oriented in all directions, fixes carbon in a central pyrenoid and produces starch in peripheral protuberances. Analysis of cells kept in the dark shows that starch is a short-term carbon storage. The biogenesis of cytosolic droplets shows that they are loaded with triacylglycerol and carotenoids for long-term carbon storage and protection against oxidative stress.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell w multi-organelle?\n",
      "Stop? (Y/N)N\n",
      "37985241 Title:  Probing the in situ volumes of Arabidopsis leaf plastids using three-dimensional confocal and scanning electron microscopy\n",
      "Abstract:  ['Leaf plastids harbor a plethora of biochemical reactions including photosynthesis, one of the most important metabolic pathways on Earth. Scientists are eager to unveil the physiological processes within the organelle but also their interconnection with the rest of the plant cell. An increasingly important feature of this venture is to use experimental data in the design of metabolic models. A remaining obstacle has been the limited in situ volume information of plastids and other cell organelles. To fill this gap for chloroplasts, we established three microscopy protocols delivering in situ volumes based on: (i) chlorophyll fluorescence emerging from the thylakoid membrane, (ii) a CFP marker embedded in the envelope, and (iii) calculations from serial block-face scanning electron microscopy (SBFSEM). The obtained data were corroborated by comparing wild-type data with two mutant lines affected in the plastid division machinery known to produce small and large mesophyll chloroplasts, respectively. Furthermore, we also determined the volume of the much smaller guard cell plastids. Interestingly, their volume is not governed by the same components of the division machinery which defines mesophyll plastid size. Based on our three approaches, the average volume of a mature Col-0 wild-type mesophyll chloroplasts is 93\\u2009μm<sup>3</sup> . Wild-type guard cell plastids are approximately 18\\u2009μm<sup>3</sup> . Lastly, our comparative analysis shows that the chlorophyll fluorescence analysis can accurately determine chloroplast volumes, providing an important tool to research groups without access to transgenic marker lines expressing genetically encoded fluorescence proteins or costly SBFSEM equipment.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: single-organelle scale?\n",
      "Stop? (Y/N)N\n",
      "37996434 Title:  3D reconstructions of parasite development and the intracellular niche of the microsporidian pathogen Encephalitozoon intestinalis\n",
      "Abstract:  ['Microsporidia are an early-diverging group of fungal pathogens with a wide host range. Several microsporidian species cause opportunistic infections in humans that can be fatal. As obligate intracellular parasites with highly reduced genomes, microsporidia are dependent on host metabolites for successful replication and development. Our knowledge of microsporidian intracellular development remains rudimentary, and our understanding of the intracellular niche occupied by microsporidia has relied on 2D TEM images and light microscopy. Here, we use serial block-face scanning electron microscopy (SBF-SEM) to capture 3D snapshots of the human-infecting species, Encephalitozoon intestinalis, within host cells. We track E. intestinalis development through its life cycle, which allows us to propose a model for how its infection organelle, the polar tube, is assembled de novo in developing spores. 3D reconstructions of parasite-infected cells provide insights into the physical interactions between host cell organelles and parasitophorous vacuoles, which contain the developing parasites. The host cell mitochondrial network is substantially remodeled during E. intestinalis infection, leading to mitochondrial fragmentation. SBF-SEM analysis shows changes in mitochondrial morphology in infected cells, and live-cell imaging provides insights into mitochondrial dynamics during infection. Our data provide insights into parasite development, polar tube assembly, and microsporidia-induced host mitochondria remodeling.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments:  \n",
      "Stop? (Y/N)N\n",
      "38014052 Title:  Multiplexed 3D Analysis of Immune States and Niches in Human Tissue\n",
      "Abstract:  ['Tissue homeostasis and the emergence of disease are controlled by changes in the proportions of resident and recruited cells, their organization into cellular neighbourhoods, and their interactions with acellular tissue components. Highly multiplexed tissue profiling (spatial omics)<sup>1</sup> makes it possible to study this microenvironment <i>in situ</i>, usually in 4-5 micron thick sections (the standard histopathology format)<sup>2</sup>. Microscopy-based tissue profiling is commonly performed at a resolution sufficient to determine cell types but not to detect subtle morphological features associated with cytoskeletal reorganisation, juxtracrine signalling, or membrane trafficking<sup>3</sup>. Here we describe a high-resolution 3D imaging approach able to characterize a wide variety of organelles and structures at sub-micron scale while simultaneously quantifying millimetre-scale spatial features. This approach combines cyclic immunofluorescence (CyCIF) imaging<sup>4</sup> of over 50 markers with confocal microscopy of archival human tissue thick enough (30-40 microns) to fully encompass two or more layers of intact cells. 3D imaging of entire cell volumes substantially improves the accuracy of cell phenotyping and allows cell proximity to be scored using plasma membrane apposition, not just nuclear position. In pre-invasive melanoma <i>in situ</i><sup>5</sup>, precise phenotyping shows that adjacent melanocytic cells are plastic in state and participate in tightly localised niches of interferon signalling near sites of initial invasion into the underlying dermis. In this and metastatic melanoma, mature and precursor T cells engage in an unexpectedly diverse array of juxtracrine and membrane-membrane interactions as well as looser \"neighbourhood\" associations<sup>6</sup> whose morphologies reveal functional states. These data provide new insight into the transitions occurring during early tumour formation and immunoediting and demonstrate the potential for phenotyping of tissues at a level of detail previously restricted to cultured cells and organoids.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole single cells, resolvable in tissue context?\n",
      "Stop? (Y/N)N\n",
      "38081848 Title:  Protein and lipid expansion microscopy with trypsin and tyramide signal amplification for 3D imaging\n",
      "Abstract:  ['Expansion microscopy, whereby the relative positions of biomolecules are physically increased via hydrogel expansion, can be used to reveal ultrafine structures of cells under a conventional microscope. Despite its utility for achieving super-resolution imaging, expansion microscopy suffers a major drawback, namely reduced fluorescence signals caused by excessive proteolysis and swelling effects. This caveat results in a lower photon budget and disfavors fluorescence imaging over a large field of view that can cover an entire expanded cell, especially in 3D. In addition, the complex procedures and specialized reagents of expansion microscopy hinder its popularization. Here, we modify expansion microscopy by deploying trypsin digestion to reduce protein loss and tyramide signal amplification to enhance fluorescence signal for point-scanning-based imaging. We name our new methodology TT-ExM to indicate dual trypsin and tyramide treatments. TT-ExM may be applied for both antibody and lipid staining. TT-ExM displayed enhanced protein retention for endoplasmic reticulum and mitochondrial markers in COS-7 cell cultures. Importantly, TT-ExM-based lipid staining clearly revealed the complex 3D membrane structures in entire expanded cells. Through combined lipid and DNA staining, our TT-ExM methodology highlighted mitochondria by revealing their DNA and membrane structures in cytoplasm, as well as the lipid-rich structures formed via phase separation in nuclei at interphase. We also observed lipid-rich chromosome matrices in the mitotic cells. These high-quality 3D images demonstrate the practicality of TT-ExM. Thus, readily available reagents can be deployed in TT-ExM to significantly enhance fluorescence signals and generate high-quality and ultrafine-resolution images under confocal microscopy.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: 3D/whole cell/multi-organelle reconstructions?\n",
      "Stop? (Y/N)N\n",
      "38132834 Title:  Acousto-optofluidic 3D single cell imaging of macrophage phagocytosis of Pseudomonas Aeruginosa\n",
      "Abstract:  ['Understanding how immune cells such as monocytes or macrophages within our blood and tissue engulf and destroy foreign organisms is important for developing new therapies. The process undertaken by these cells, called phagocytosis, has yet to be observed in real-time at the single cell level. Microfluidic-based imaging platforms offer a wide range of tools for precise fluid control and biomolecule manipulation that makes regulating long term experiments and data collection possible. With the compatibility between acoustofluidics and light-sheet fluorescent microscopy (LSFM) previously demonstrated, here an acousto-optfluidic device with on-chip fluid flow direction control was developed. The standing surface acoustic waves (SSAWs) were used to trap, load and safeguard individual cells within a highly controllable fluid loop, created <i>via</i> the triggering of on-chip PDMS valves, to demonstrate multiple rounds of live single cell imaging. The valves allowed for the direction of the fluid flow to be changed (between forward and reverse operation) without altering the inlet flow rate, an important factor for performing reproducible and comparable imaging of samples over time. With this high-resolution imaging system, volumetric reconstructions of phagocytosed bacteria within macrophages could be resolved over a total of 9 rounds of imaging: totalling 19 reconstructed images of the cell membrane with visible intracellular bacteria.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: organelles?\n",
      "Stop? (Y/N)N\n",
      "38148303 Title:  Three-Dimensional Ultrastructure of Arabidopsis Cotyledons Infected with Colletotrichum higginsianum\n",
      "Abstract:  ['We used serial block-face scanning electron microscopy (SBF-SEM) to study the host-pathogen interface between <i>Arabidopsis</i> cotyledons and the hemibiotrophic fungus <i>Colletotrichum higginsianum</i>. By combining high-pressure freezing and freeze-substitution with SBF-SEM, followed by segmentation and reconstruction of the imaging volume using the freely accessible software IMOD, we created 3D models of the series of cytological events that occur during the <i>Colletotrichum-Arabidopsis</i> susceptible interaction. We found that the host cell membranes underwent massive expansion to accommodate the rapidly growing intracellular hypha. As the fungal infection proceeded from the biotrophic to the necrotrophic stage, the host cell membranes went through increasing levels of disintegration culminating in host cell death. Intriguingly, we documented autophagosomes in proximity to biotrophic hyphae using transmission electron microscopy (TEM) and a concurrent increase in autophagic flux between early to mid/late biotrophic phase of the infection process. Occasionally, we observed osmiophilic bodies in the vicinity of biotrophic hyphae using TEM only and near necrotrophic hyphae under both TEM and SBF-SEM. Overall, we established a method for obtaining serial SBF-SEM images, each with a lateral (<i>x-y</i>) pixel resolution of 10 nm and an axial (<i>z</i>) resolution of 40 nm, that can be reconstructed into interactive 3D models using the IMOD. Application of this method to the <i>Colletotrichum-Arabidopsis</i> pathosystem allowed us to more fully understand the spatial arrangement and morphological architecture of the fungal hyphae after they penetrate epidermal cells of <i>Arabidopsis</i> cotyledons and the cytological changes the host cell undergoes as the infection progresses toward necrotrophy. [Formula: see text] Copyright © 2024 The Author(s). This is an open access article distributed under the CC BY 4.0 International license.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: multi-organelle/whole cell?\n",
      "Stop? (Y/N)N\n",
      "38158370 Title:  Advanced Imaging Techniques for the Characterization of Subcellular Organelle Structure in Pancreatic Islet β Cells\n",
      "Abstract:  ['Type 2 diabetes (T2D) affects more than 32.3 million individuals in the United States, creating an economic burden of nearly $966 billion in 2021. T2D results from a combination of insulin resistance and inadequate insulin secretion from the pancreatic β cell. However, genetic and physiologic data indicate that defects in β cell function are the chief determinant of whether an individual with insulin resistance will progress to a diagnosis of T2D. The subcellular organelles of the insulin secretory pathway, including the endoplasmic reticulum, Golgi apparatus, and secretory granules, play a critical role in maintaining the heavy biosynthetic burden of insulin production, processing, and secretion. In addition, the mitochondria enable the process of insulin release by integrating the metabolism of nutrients into energy output. Advanced imaging techniques are needed to determine how changes in the structure and composition of these organelles contribute to the loss of insulin secretory capacity in the β cell during T2D. Several microscopy techniques, including electron microscopy, fluorescence microscopy, and soft X-ray tomography, have been utilized to investigate the structure-function relationship within the β cell. In this overview article, we will detail the methodology, strengths, and weaknesses of each approach. © 2024 American Physiological Society. Compr Physiol 14:5243-5267, 2024.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: overview article (but keep to read)\n",
      "Stop? (Y/N)N\n",
      "38162124 Title:  Segmentation of cellular ultrastructures on sparsely labeled 3D electron microscopy images using deep learning\n",
      "Abstract:  ['Focused ion beam-scanning electron microscopy (FIB-SEM) images can provide a detailed view of the cellular ultrastructure of tumor cells. A deeper understanding of their organization and interactions can shed light on cancer mechanisms and progression. However, the bottleneck in the analysis is the delineation of the cellular structures to enable quantitative measurements and analysis. We mitigated this limitation using deep learning to segment cells and subcellular ultrastructure in 3D FIB-SEM images of tumor biopsies obtained from patients with metastatic breast and pancreatic cancers. The ultrastructures, such as nuclei, nucleoli, mitochondria, endosomes, and lysosomes, are relatively better defined than their surroundings and can be segmented with high accuracy using a neural network trained with sparse manual labels. Cell segmentation, on the other hand, is much more challenging due to the lack of clear boundaries separating cells in the tissue. We adopted a multi-pronged approach combining detection, boundary propagation, and tracking for cell segmentation. Specifically, a neural network was employed to detect the intracellular space; optical flow was used to propagate cell boundaries across the z-stack from the nearest ground truth image in order to facilitate the separation of individual cells; finally, the filopodium-like protrusions were tracked to the main cells by calculating the intersection over union measure for all regions detected in consecutive images along z-stack and connecting regions with maximum overlap. The proposed cell segmentation methodology resulted in an average Dice score of 0.93. For nuclei, nucleoli, and mitochondria, the segmentation achieved Dice scores of 0.99, 0.98, and 0.86, respectively. The segmentation of FIB-SEM images will enable interpretative rendering and provide quantitative image features to be associated with relevant clinical variables.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: cell segmentation sufficient for individual cell reconstructions?\n",
      "Stop? (Y/N)N\n",
      "38198284 Title:  Near-Native Imaging of Label-Free Silver Nanoparticles-Triggered 3D Subcellular Ultrastructural Reorganization in Microalgae\n",
      "Abstract:  ['Understanding the spatial orientation of nanoparticles and the corresponding subcellular architecture events favors uncovering fundamental toxic mechanisms and predicting response pathways of organisms toward environmental stressors. Herein, we map the spatial location of label-free citrate-coated Ag nanoparticles (Cit-AgNPs) and the corresponding subcellular reorganization in microalgae by a noninvasive 3D imaging approach, cryo-soft X-ray tomography (cryo-SXT). Cryo-SXT near-natively displays the 3D maps of Cit-AgNPs presenting in rarely identified sites, namely, extracellular polymeric substances (EPS) and the cytoplasm. By comparative 3D morphological assay, we observe that Cit-AgNPs disrupt the cellular ultrastructural homeostasis, triggering a severe malformation of cytoplasmic organelles with energy-producing and stress-regulating functions. AgNPs exposure causes evident disruption of the chloroplast membrane, significant attenuation of the pyrenoid matrix and starch sheath, extreme swelling of starch granules and lipid droplets, and shrinkage of the nucleolus. In accompaniment, the number and volume occupancy of starch granules are significantly increased. Meanwhile, the spatial topology of starch granules extends from the chloroplast to the cytoplasm with a dispersed distribution. Linking the dynamics of the internal structure and the alteration of physiological properties, we derive a comprehensive cytotoxic and response pathway of microalgae exposed to AgNPs. This work provides a perspective for assessing the toxicity at subcellular scales to achieve label-free nanoparticle-caused ultrastructure remodeling of phytoplankton.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments:  \n",
      "Stop? (Y/N)N\n",
      "38232737 Title:  Using MitER for 3D analysis of mitochondrial morphology and ER contacts\n",
      "Abstract:  ['We have developed an open-source workflow that allows for quantitative single-cell analysis of organelle morphology, distribution, and inter-organelle contacts with an emphasis on the analysis of mitochondria and mitochondria-endoplasmic reticulum (mito-ER) contact sites. As the importance of inter-organelle contacts becomes more widely recognized, there is a concomitant increase in demand for tools to analyze subcellular architecture. Here, we describe a workflow we call MitER (pronounced \"mightier\"), which allows for automated calculation of organelle morphology, distribution, and inter-organelle contacts from 3D renderings by employing the animation software Blender. We then use MitER to quantify the variations in the mito-ER networks of Saccharomyces cerevisiae, revealing significantly more mito-ER contacts within respiring cells compared to fermenting cells. We then demonstrate how this workflow can be applied to mammalian systems and used to monitor mitochondrial dynamics and inter-organelle contact in time-lapse studies.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell?\n",
      "Stop? (Y/N)N\n",
      "38315771 Title:  Stereochemistry-Dependent Labeling of Organelles with a Near-Infrared-Emissive Phosphorus-Bridged Rhodamine Dye in Live-Cell Imaging\n",
      "Abstract:  ['The development of near-infrared (NIR) fluorophores that have both excellent chemical stability and photostability, as well as efficient cell permeability, is highly demanded. In this study, we present phospha-rhodamine (POR) dyes which display significantly improved performance for protein labeling. This is achieved by incorporating a 2-carboxy-3-benzothiophenyl group at the 9-position of the xanthene scaffold. The resulting cis and trans isomers were successfully isolated and structurally characterized using X-ray diffraction. The HaloTag ligand conjugates of the two isomers exhibited different staining abilities in live cells. While the cis isomer showed non-specific accumulation on the organelle membranes, the trans isomer selectively labeled the HaloTag-fused proteins, enabling the long-term imaging of cell division and the 5-color imaging of cell organelles. Molecular dynamics simulations of the HaloTag ligand conjugates within the lipid membrane suggested that the cis isomer is more prone to forming oligomers in the membrane. In contrast, the oligomerization of the trans isomer is effectively suppressed by its interaction with the lipid molecules. By taking advantage of the superior labeling performance of the trans isomer and its NIR-emissive properties, multi-color time-lapse super-resolution 3D imaging, namely super-resolution 5D-imaging, of the interconnected network between the endoplasmic reticulum and microtubules was achieved in living cells.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell/3D?\n",
      "Stop? (Y/N)N\n",
      "38352445 Title:  Detailing organelle division and segregation in Plasmodium falciparum\n",
      "Abstract:  ['The malaria causing parasite, <i>Plasmodium falciparum</i>, replicates through a tightly orchestrated process termed schizogony, where approximately 32 daughter parasites are formed in a single infected red blood cell and thousands of daughter cells in mosquito or liver stages. One-per-cell organelles, such as the mitochondrion and apicoplast, need to be properly divided and segregated to ensure a complete set of organelles per daughter parasites. Although this is highly essential, details about the processes and mechanisms involved remain unknown. We developed a new reporter parasite line that allows visualization of the mitochondrion in blood and mosquito stages. Using high-resolution 3D-imaging, we found that the mitochondrion orients in a cartwheel structure, prior to stepwise, non-geometric division during the last stage of schizogony. Analysis of focused ion beam scanning electron microscopy (FIB-SEM) data confirmed these mitochondrial division stages. Furthermore, these data allowed us to elucidate apicoplast division steps, highlighted its close association with the mitochondrion, and showed putative roles of the centriolar plaques (CPs) in apicoplast segregation. These observations form the foundation for a new detailed mechanistic model of mitochondrial and apicoplast division and segregation during <i>P. falciparum</i> schizogony and pave the way for future studies into the proteins and protein complexes involved in organelle division and segregation.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments: borderline; preprint (?)\n",
      "Stop? (Y/N)N\n",
      "38367824 Title:  ColabSeg: An interactive tool for editing, processing, and visualizing membrane segmentations from cryo-ET data\n",
      "Abstract:  ['Cellular cryo-electron tomography (cryo-ET) has emerged as a key method to unravel the spatial and structural complexity of cells in their near-native state at unprecedented molecular resolution. To enable quantitative analysis of the complex shapes and morphologies of lipid membranes, the noisy three-dimensional (3D) volumes must be segmented. Despite recent advances, this task often requires considerable user intervention to curate the resulting segmentations. Here, we present ColabSeg, a Python-based tool for processing, visualizing, editing, and fitting membrane segmentations from cryo-ET data for downstream analysis. ColabSeg makes many well-established algorithms for point-cloud processing easily available to the broad community of structural biologists for applications in cryo-ET through its graphical user interface (GUI). We demonstrate the usefulness of the tool with a range of use cases and biological examples. Finally, for a large Mycoplasma pneumoniae dataset of 50 tomograms, we show how ColabSeg enables high-throughput membrane segmentation, which can be used as valuable training data for fully automated convolutional neural network (CNN)-based segmentation.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: organelles? whole-cell reconstructions?\n",
      "Stop? (Y/N)N\n",
      "38411803 Title:  Electron Microscopy Techniques for 3D Plant ER Imaging\n",
      "Abstract:  ['The endoplasmic reticulum (ER) forms an extensive network in plant cells. In leaf cells and vacuolated root cells it is mainly restricted to the cortex, whereas in the root meristem the cortical and cytoplasmic ER takes up a large volume throughout the entire cell. Only 3D electron microscopy provides sufficient resolution to understand the spatial organization of the ER in the root. Here we present two protocols for 3D EM imaging of the ER across a range of scales. For large-scale ER structure analysis, we describe selective ER staining with ZIO that allows for automated or semi-automated ER segmentation. For smaller regions of ER, we describe high-pressure freezing, which enables almost instantaneous fixation of plant tissues but without organelle specific staining. These fixation and staining techniques are suitable for a range of imaging modalities, including serial sections, array tomography, serial block face-scanning electron microscopy (SBF-SEM), or focused ion beam (FIB) SEM.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: single whole-cell scale?\n",
      "Stop? (Y/N)N\n",
      "38411805 Title:  Preparation and Imaging of Specialized ER Using Super-Resolution and TEM Techniques\n",
      "Abstract:  ['The plant endoplasmic reticulum (ER) forms several specialized structures. These include the sieve element reticulum (SER) and the desmotubule formed as the ER passes through plasmodesmata. Imaging both of these structures has been inhibited by the resolution limits of light microscopy and their relatively inaccessible locations, combined with the fragile nature of the ER. Here we describe methods to view desmotubules in live cells under 3D-structured illumination microscopy (3D-SIM) and methods to fix and prepare phloem tissue for both 3D-SIM and transmission electron microscopy (TEM), which preserve the fragile structure and allow the detailed imaging of the SER.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell/multi-organelle?\n",
      "Stop? (Y/N)N\n",
      "38411806 Title:  Quantitation of ER Morphology and Dynamics\n",
      "Abstract:  ['The plant endoplasmic reticulum forms a network of tubules connected by three-way junctions or sheet-like cisternae. Although the network is three-dimensional, in many plant cells, it is constrained to thin volume sandwiched between the vacuole and plasma membrane, effectively restricting it to a 2-D planar network. The structure of the network, and the morphology of the tubules and cisternae can be automatically extracted following intensity-independent edge-enhancement and various segmentation techniques to give an initial pixel-based skeleton, which is then converted to a graph representation. ER dynamics can be determined using optical flow techniques from computer vision or persistency analysis. Collectively, this approach yields a wealth of quantitative metrics for ER structure and can be used to describe the effects of pharmacological treatments or genetic manipulation. The software is publicly available.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole-cell?\n",
      "Stop? (Y/N)N\n",
      "38411828 Title:  Automated Segmentation and 3D Reconstruction of Different Membranes from Confocal Z-Stacks\n",
      "Abstract:  ['Confocal laser scanning microscopy (CLSM) is an advanced microscopy technique based on fluorescence technology which produces sharp images of a specimen in a single focal plane. The optical sectioning by CLSM allows to have z-stacks which can be further processed into 3D reconstructions. These then provide the option of variable perspectives and additional precise data evaluation on structural and anatomical alterations. Here, we used CLSM to image the thylakoids of cyanobacteria and the endoplasmic reticulum (ER) in moss protonemata as an example. Then, out of the confocal z-stacks, we create 3D constructions of the membranes and their alterations to present a holistic, structural view from different angles.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: not multi-organelle\n",
      "Stop? (Y/N)N\n",
      "38411826 Title:  Observing ER Dynamics over Long Timescales Using Light Sheet Fluorescence Microscopy\n",
      "Abstract:  ['The recent significant progress in developmental bio-imaging of live multicellular organisms has been greatly facilitated by the development of light sheet fluorescence microscopy (LSFM). Both commercial and custom LSFM systems offer the best means for long-term rapid data collection over a wide field of view at single-cell resolution. This is thanks to the low light exposure required for imaging and consequent limited photodamage to the biological sample, and the development of custom holders and mounting techniques that allow for specimens to be imaged in near-normal physiological conditions. This method has been successfully applied to plant cell biology and is currently seen as one of the most efficient techniques for 3D time-lapse imaging for quantitative studies. LSFM allows one to capture and quantify dynamic processes across various levels, from plant subcellular compartments to whole cells, tissues, and entire plant organs. Here we present a method to carry out LSFM on Arabidopsis leaves expressing fluorescent markers targeted to the ER. We will focus on a protocol to mount the sample, test the phototoxicity of the LSFM system, set up a LSFM experiment, and monitor the dynamics of the ER during heat shock.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: multi-organelle? (maybe nucleus visible)\n",
      "Stop? (Y/N)N\n",
      "38416776 Title:  Whole cell reconstructions of Leishmania mexicana through the cell cycle\n",
      "Abstract:  ['The unicellular parasite Leishmania has a precisely defined cell architecture that is inherited by each subsequent generation, requiring a highly coordinated pattern of duplication and segregation of organelles and cytoskeletal structures. A framework of nuclear division and morphological changes is known from light microscopy, yet this has limited resolution and the intrinsic organisation of organelles within the cell body and their manner of duplication and inheritance is unknown. Using volume electron microscopy approaches, we have produced three-dimensional reconstructions of different promastigote cell cycle stages to give a spatial and quantitative overview of organelle positioning, division and inheritance. The first morphological indications seen in our dataset that a new cell cycle had begun were the assembly of a new flagellum, the duplication of the contractile vacuole and the increase in volume of the nucleus and kinetoplast. We showed that the progression of the cytokinesis furrow created a specific pattern of membrane indentations, while our analysis of sub-pellicular microtubule organisation indicated that there is likely a preferred site of new microtubule insertion. The daughter cells retained these indentations in their cell body for a period post-abscission. By comparing cultured and sand fly derived promastigotes, we found an increase in the number and overall volume of lipid droplets in the promastigotes from the sand fly, reflecting a change in their metabolism to ensure transmissibility to the mammalian host. Our insights into the cell cycle mechanics of Leishmania will support future molecular cell biology analyses of these parasites.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments:  \n",
      "Stop? (Y/N)N\n",
      "38429930 Title:  Reorganization of mitochondria-organelle interactions during postnatal development in skeletal muscle\n",
      "Abstract:  ['Skeletal muscle cellular development requires the integrated assembly of mitochondria and other organelles adjacent to the sarcomere in support of muscle contractile performance. However, it remains unclear how interactions among organelles and with the sarcomere relates to the development of muscle cell function. Here, we combine 3D volume electron microscopy, proteomic analyses, and live cell functional imaging to investigate the postnatal reorganization of mitochondria-organelle interactions in skeletal muscle. We show that while mitochondrial networks are disorganized and loosely associated with the contractile apparatus at birth, contact sites among mitochondria, lipid droplets and the sarcoplasmic reticulum are highly abundant in neonatal muscles. The maturation process is characterized by a transition to highly organized mitochondrial networks wrapped tightly around the muscle sarcomere but also to less frequent interactions with both lipid droplets and the sarcoplasmic reticulum. Concomitantly, expression of proteins involved in mitochondria-organelle membrane contact sites decreases during postnatal development in tandem with a decrease in abundance of proteins associated with sarcomere assembly despite an overall increase in contractile protein abundance. Functionally, parallel measures of mitochondrial membrane potential, NADH redox status, and NADH flux within intact cells revealed that mitochondria in adult skeletal muscle fibres maintain a more activated electron transport chain compared with neonatal muscle mitochondria. These data demonstrate a developmental redesign reflecting a shift from muscle cell assembly and frequent inter-organelle communication toward a muscle fibre with mitochondrial structure, interactions, composition and function specialized to support contractile function. KEY POINTS: Mitochondrial network organization is remodelled during skeletal muscle postnatal development. The mitochondrial outer membrane is in frequent contact with other organelles at birth and transitions to more close associations with the contractile apparatus in mature muscles. Mitochondrial energy metabolism becomes more activated during postnatal development. Understanding the developmental redesign process within skeletal muscle cells may help pinpoint specific areas of deficit in muscles with developmental disorders.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole-cell? borderline?\n",
      "Stop? (Y/N)N\n",
      "38438356 Title:  Light-field flow cytometry for high-resolution, volumetric and multiparametric 3D single-cell analysis\n",
      "Abstract:  ['Imaging flow cytometry (IFC) combines flow cytometry and fluorescence microscopy to enable high-throughput, multiparametric single-cell analysis with rich spatial details. However, current IFC techniques remain limited in their ability to reveal subcellular information with a high 3D resolution, throughput, sensitivity, and instrumental simplicity. In this study, we introduce a light-field flow cytometer (LFC), an IFC system capable of high-content, single-shot, and multi-color acquisition of up to 5,750 cells per second with a near-diffraction-limited resolution of 400-600\\u2009nm in all three dimensions. The LFC system integrates optical, microfluidic, and computational strategies to facilitate the volumetric visualization of various 3D subcellular characteristics through convenient access to commonly used epi-fluorescence platforms. We demonstrate the effectiveness of LFC in assaying, analyzing, and enumerating intricate subcellular morphology, function, and heterogeneity using various phantoms and biological specimens. The advancement offered by the LFC system presents a promising methodological pathway for broad cell biological and translational discoveries, with the potential for widespread adoption in biomedical research.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole-cell?\n",
      "Stop? (Y/N)N\n",
      "38471506 Title:  Correlative cryo-soft X-ray tomography and cryo-structured illumination microscopy reveal changes to lysosomes in amyloid-β-treated neurons\n",
      "Abstract:  [\"Protein misfolding is common to neurodegenerative diseases (NDs) including Alzheimer's disease (AD), which is partly characterized by the self-assembly and accumulation of amyloid-beta in the brain. Lysosomes are a critical component of the proteostasis network required to degrade and recycle material from outside and within the cell and impaired proteostatic mechanisms have been implicated in NDs. We have previously established that toxic amyloid-beta oligomers are endocytosed, accumulate in lysosomes, and disrupt the endo-lysosomal system in neurons. Here, we use pioneering correlative cryo-structured illumination microscopy and cryo-soft X-ray tomography imaging techniques to reconstruct 3D cellular architecture in the native state revealing reduced X-ray density in lysosomes and increased carbon dense vesicles in oligomer treated neurons compared with untreated cells. This work provides unprecedented visual information on the changes to neuronal lysosomes inflicted by amyloid beta oligomers using advanced methods in structural cell biology.\"]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell reconstruction?\n",
      "Stop? (Y/N)N\n",
      "38501891 Title:  Automated segmentation of cell organelles in volume electron microscopy using deep learning\n",
      "Abstract:  ['Recent advances in computing power triggered the use of artificial intelligence in image analysis in life sciences. To train these algorithms, a large enough set of certified labeled data is required. The trained neural network is then capable of producing accurate instance segmentation results that will then need to be re-assembled into the original dataset: the entire process requires substantial expertise and time to achieve quantifiable results. To speed-up the process, from cell organelle detection to quantification across electron microscopy modalities, we propose a deep-learning based approach for fast automatic outline segmentation (FAMOUS), that involves organelle detection combined with image morphology, and 3D meshing to automatically segment, visualize and quantify cell organelles within volume electron microscopy datasets. From start to finish, FAMOUS provides full segmentation results within a week on previously unseen datasets. FAMOUS was showcased on a HeLa cell dataset acquired using a focused ion beam scanning electron microscope, and on yeast cells acquired by transmission electron tomography. RESEARCH HIGHLIGHTS: Introducing a rapid, multimodal machine-learning workflow for the automatic segmentation of 3D cell organelles. Successfully applied to a variety of volume electron microscopy datasets and cell lines. Outperforming manual segmentation methods in time and accuracy. Enabling high-throughput quantitative cell biology.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell/multi-organelle?\n",
      "Stop? (Y/N)N\n",
      "38552613 Title:  Unraveling cellular complexity with transient adapters in highly multiplexed super-resolution imaging\n",
      "Abstract:  ['Mapping the intricate spatial relationships between the many different molecules inside a cell is essential to understanding cellular functions in all their complexity. Super-resolution fluorescence microscopy offers the required spatial resolution but struggles to reveal more than four different targets simultaneously. Exchanging labels in subsequent imaging rounds for multiplexed imaging extends this number but is limited by its low throughput. Here, we present a method for rapid multiplexed super-resolution microscopy that can, in principle, be applied to a nearly unlimited number of molecular targets by leveraging fluorogenic labeling in conjunction with transient adapter-mediated switching for high-throughput DNA-PAINT (FLASH-PAINT). We demonstrate the versatility of FLASH-PAINT with four applications: mapping nine proteins in a single mammalian cell, elucidating the functional organization of primary cilia by nine-target imaging, revealing the changes in proximity of thirteen different targets in unperturbed and dissociated Golgi stacks, and investigating and quantifying inter-organelle contacts at 3D super-resolution.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell/multi-organelle?\n",
      "Stop? (Y/N)N\n",
      "38559764 Title:  Zygospore development of Spirogyra (Charophyta) investigated by serial block-face scanning electron microscopy and 3D reconstructions\n",
      "Abstract:  [\"Sexual reproduction of Zygnematophyceae by conjugation is a less investigated topic due to the difficulties of the induction of this process and zygospore ripening under laboratory conditions. For this study, we collected field sampled zygospores of <i>Spirogyra mirabilis</i> and three additional <i>Spirogyra</i> strains in Austria and Greece. Serial block-face scanning electron microscopy was performed on high pressure frozen and freeze substituted zygospores and 3D reconstructions were generated, allowing a comprehensive insight into the process of zygospore maturation, involving storage compound and organelle rearrangements. Chloroplasts are drastically changed, while young stages contain both parental chloroplasts, the male chloroplasts are aborted and reorganised as 'secondary vacuoles' which initially contain plastoglobules and remnants of thylakoid membranes. The originally large pyrenoids and the volume of starch granules is significantly reduced during maturation (young: 8 ± 5 µm³, mature: 0.2 ± 0.2 µm³). In contrast, lipid droplets (LDs) increase significantly in number upon zygospore maturation, while simultaneously getting smaller (young: 21 ± 18 µm³, mature: 0.1 ± 0.2 and 0.5 ± 0.9 µm³). Only in <i>S. mirabilis</i> the LD volume increases (34 ± 29 µm³), occupying ~50% of the zygospore volume. Mature zygospores contain barite crystals as confirmed by Raman spectroscopy with a size of 0.02 - 0.05 µm³. The initially thin zygospore cell wall (~0.5 µm endospore, ~0.8 µm exospore) increases in thickness and develops a distinct, electron dense mesospore, which has a reticulate appearance (~1.4 µm) in <i>Spirogyra</i> sp. from Greece. The exo- and endospore show cellulose microfibrils in a helicoidal pattern. In the denser endospore, pitch angles of the microfibril layers were calculated: ~18 ± 3° in <i>S. mirabilis</i>, ~20 ± 3° in <i>Spirogyra</i> sp. from Austria and ~38 ± 8° in <i>Spirogyra</i> sp. from Greece. Overall this study gives new insights into <i>Spirogyra</i> sp. zygospore development, crucial for survival during dry periods and dispersal of this genus.\"]\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole-cell? (PMC figs show half-cell reconstructions; maybe whole-cells are in supp)\n",
      "Stop? (Y/N)N\n",
      "38564214 Title:  Impact of capillary and sarcolemmal proximity on mitochondrial structure and energetic function in skeletal muscle\n",
      "Abstract:  ['Mitochondria within skeletal muscle cells are located either between the muscle contractile apparatus (interfibrillar mitochondria, IFM) or beneath the cell membrane (subsarcolemmal mitochondria, SSM), with several structural and functional differences reported between IFM and SSM. However, recent 3D imaging studies demonstrate that mitochondria are particularly concentrated in the proximity of capillaries embedded in sarcolemmal grooves rather than in proximity to the sarcolemma itself (paravascular mitochondria, PVM). To evaluate the impact of capillary vs. sarcolemmal proximity, we compared the structure and function of skeletal muscle mitochondria located either lateral to embedded capillaries (PVM), adjacent to the sarcolemma but not in PVM pools (SSM) or interspersed between sarcomeres (IFM). Mitochondrial morphology and interactions were assessed by 3D electron microscopy coupled with machine learning segmentation, whereas mitochondrial energy conversion was assessed by two-photon microscopy of mitochondrial membrane potential, content, calcium, NADH redox and flux in live, intact cells. Structurally, although PVM and SSM were similarly larger than IFM, PVM were larger, rounder and had more physical connections to neighbouring mitochondria compared to both IFM and SSM. Functionally, PVM had similar or greater basal NADH flux compared to SSM and IFM, respectively, despite a more oxidized NADH pool and a greater membrane potential, signifying a greater activation of the electron transport chain in PVM. Together, these data indicate that proximity to capillaries has a greater impact on resting mitochondrial energy conversion and distribution in skeletal muscle than the sarcolemma alone. KEY POINTS: Capillaries have a greater impact on mitochondrial energy conversion in skeletal muscle than the sarcolemma. Paravascular mitochondria are larger, and the outer mitochondrial membrane is more connected with neighbouring mitochondria. Interfibrillar mitochondria are longer and have greater contact sites with other organelles (i.e. sarcoplasmic reticulum and lipid droplets). Paravascular mitochondria have greater activation of oxidative phosphorylation than interfibrillar mitochondria at rest, although this is not regulated by calcium.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell? (probably not)\n",
      "Stop? (Y/N)N\n",
      "38590054 Title:  Three-dimensional ultrastructural and anatomical analysis of prostatic neuroendocrine cells in mice\n",
      "Abstract:  [StringElement('A few studies have examined the ultrastructure of prostatic neuroendocrine cells (NECs), and no study has focused on their ultrastructure in three dimensions. In this study, three-dimensional ultrastructural analysis of mouse prostatic NECs was performed to clarify their anatomical characteristics.', attributes={'Label': 'BACKGROUND', 'NlmCategory': 'BACKGROUND'}), StringElement('Three 13-week-old male C57BL/6 mice were deeply anesthetized, perfused with physiological saline and 2% paraformaldehyde, and then placed in 2.5% glutaraldehyde in 0.1\\u2009M cacodylate (pH 7.3) buffer for electron microscopy. After perfusion, the lower urinary tract, which included the bladder, prostate, coagulation gland, seminal vesicle, upper vas deferens, and urethra, was removed, and the specimen was cut into small cubes and subjected to postfixation and en bloc staining. Three-dimensional ultrastructural analysis was performed on NECs, the surrounding cells, tissues, and nerves using focused ion beam/scanning electron microscope tomography.', attributes={'Label': 'METHODS', 'NlmCategory': 'METHODS'}), StringElement('Twenty-seven serial sections were used in the present study, and 32 mouse prostatic NECs were analyzed. Morphologically, the NECs could be classified into three types: flask, flat, and closed. Closed-shaped NECs were always adjacent to flask-shaped cells. The flask-shaped and flat NECs were in direct contact with the ductal lumen and always had microvilli at their contact points. Many of the NECs had accompanying nerves, some of which terminated on the surface in contact with the NEC.', attributes={'Label': 'RESULTS', 'NlmCategory': 'RESULTS'}), StringElement('Three-dimensional ultrastructural analysis of mouse prostatic NECs was performed. These cells can be classified into three types based on shape. Novel findings include the presence of microvilli at their points of contact with the ductal lumen and the presence of accompanying nerves.', attributes={'Label': 'CONCLUSIONS', 'NlmCategory': 'CONCLUSIONS'})]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell? organelles?\n",
      "Stop? (Y/N)N\n",
      "38693015 Title:  The dynamic interfaces between the ER and mitochondria revealed by 3D electron microscopy and single-molecule imaging\n",
      "No abstract available, indicate Check Full Text (C)\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: no abstract available\n",
      "Stop? (Y/N)N\n",
      "38705628 Title:  Laboratory based correlative cryo-soft X-ray tomography and cryo-fluorescence microscopy\n",
      "Abstract:  ['Cryo-soft X-ray tomography is the unique technology that can image whole intact cells in 3D under normal and pathological conditions without labelling or fixation, at high throughput and spatial resolution. The sample preparation is relatively straightforward; requiring just fast freezing of the specimen before transfer to the microscope for imaging. It is also possible to image chemically fixed samples where necessary. The technique can be correlated with cryo fluorescence microscopy to localize fluorescent proteins to organelles within the whole cell volume. Cryo-correlated light and soft X-ray tomography is particularly useful for the study of gross morphological changes brought about by disease or drugs. For example, viral fluorescent tags can be co-localized to sites of viral replication in the soft X-ray volume. In general this approach is extremely useful in the study of complex 3D organelle structure, nanoparticle uptake or in the detection of rare events in the context of whole cell structure. The main challenge of soft X-ray tomography is that the soft X-ray illumination required for imaging has heretofore only been available at a small number of synchrotron labs worldwide. Recently, a compact device with a footprint small enough to fit in a standard laboratory setting has been deployed (\"the SXT-100\") and is routinely imaging cryo prepared samples addressing a variety of disease and drug research applications. The SXT-100 facilitates greater access to this powerful technique and greatly increases the scope and throughput of potential research projects. Furthermore, the availability of cryo-soft X-ray tomography in the laboratory will accelerate the development of novel correlative and multimodal workflows by integration with light and electron microscope based approaches. It also allows for co-location of this powerful imaging modality at BSL3 labs or other facilities where safety or intellectual property considerations are paramount. Here we describe the compact SXT-100 microscope along with its novel integrated cryo-fluorescence imaging capability.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: relevant demo data? \n",
      "Stop? (Y/N)N\n",
      "38709484 Title:  A Pipeline for Dynamic Analysis of Mitochondrial Content in Developing T Cells: Bridging the Gap Between High-Throughput Flow Cytometry and Single-Cell Microscopy Analysis\n",
      "Abstract:  ['Analyzing the dynamics of mitochondrial content in developing T cells is crucial for understanding the metabolic state during T\\xa0cell development. However, monitoring mitochondrial content in real-time needs a balance of cell viability and image resolution. In this chapter, we present experimental protocols for measuring mitochondrial content in developing T cells using three modalities: bulk analysis via flow cytometry, volumetric imaging in laser scanning confocal microscopy, and dynamic live-cell monitoring in spinning disc confocal microscopy. Next, we provide an image segmentation and centroid tracking-based analysis pipeline for automated quantification of a large number of microscopy images. These protocols together offer comprehensive approaches to investigate mitochondrial dynamics in developing T cells, enabling a deeper understanding of their metabolic processes.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: focus on mitochondria; book chapter?\n",
      "Stop? (Y/N)N\n",
      "38756048 Title:  Characterization of Optical, Thermal, and Viscoelastic Properties of Pollenkitt in Angiosperm Pollen Using In-Line Digital Holographic Microscopy\n",
      "Abstract:  ['Pollen grains are remarkable material composites, with various organelles in their fragile interior protected by a strong shell made of sporopollenin. The outermost layer of angiosperm pollen grains contains a lipid-rich substance called pollenkitt, which is a natural bioadhesive that helps preserve structural integrity when the pollen grain is exposed to external environmental stresses. In addition, its viscous nature enables it to adhere to various floral and insect surfaces, facilitating the pollination process. To examine the physicochemical properties of aqueous pollenkitt droplets, we used in-line digital holographic microscopy to capture light scattering from individual pollenkitt particles. Comparison of pollenkitt holograms to those modeled using the Lorenz-Mie theory enables investigations into the minute variations in the refractive index and size resulting from changes in local temperature and pollen aging.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?E\n",
      "Comments: not whole cell, focus on pollenkitt (outer substance)\n",
      "Stop? (Y/N)N\n",
      "38786091 Title:  Subcellular Feature-Based Classification of α and β Cells Using Soft X-ray Tomography\n",
      "Abstract:  ['The dysfunction of α and β cells in pancreatic islets can lead to diabetes. Many questions remain on the subcellular organization of islet cells during the progression of disease. Existing three-dimensional cellular mapping approaches face challenges such as time-intensive sample sectioning and subjective cellular identification. To address these challenges, we have developed a subcellular feature-based classification approach, which allows us to identify α and β cells and quantify their subcellular structural characteristics using soft X-ray tomography (SXT). We observed significant differences in whole-cell morphological and organelle statistics between the two cell types. Additionally, we characterize subtle biophysical differences between individual insulin and glucagon vesicles by analyzing vesicle size and molecular density distributions, which were not previously possible using other methods. These sub-vesicular parameters enable us to predict cell types systematically using supervised machine learning. We also visualize distinct vesicle and cell subtypes using Uniform Manifold Approximation and Projection (UMAP) embeddings, which provides us with an innovative approach to explore structural heterogeneity in islet cells. This methodology presents an innovative approach for tracking biologically meaningful heterogeneity in cells that can be applied to any cellular system.']\n",
      "Exclude (E), Check Full Text (C), or Include (I)?I\n",
      "Comments:  \n",
      "Stop? (Y/N)N\n",
      "38854505 Title:  Pleomorphism in Biological Units of Life: Morphological Heterogeneity in Cells Does Not Translate Uniformly to Subcellular Components\n",
      "Abstract:  ['The interplay of the three-dimensional (3D) distribution of various subcellular components and their interactions are expected to control overall cellular morphology in biology. In this study, we aimed to determine whether the pleomorphy observed at the whole-cell level is being reflected by the components constituting the cells by focusing on the 3D distribution of pixel intensities at the single-cell level of the whole (cell) and its parts (the seven subcellular components of the cells-self-assemblies of smaller units). We rigorously acquired and analyzed the image data of RAW264.7 cells at the single-cell level. We report asymmetries in the spatial distribution of pixel intensities at the whole-cell and subcellular component levels along with the occurrence of alterations when pleomorphism is reduced by synchronization of the cell cycle. From our repertoire of seven subcellular components, we report ER, mitochondria, and tubulin to be independent of whole-cell apico-basal heterogeneity of optical density while nuclear, plasma membrane, lysosomal, and actin fluorescence distributions are found to contribute to the apico-basal polarity of the whole cell. While doing so, we have also developed an image analysis algorithm utilizing 2D segmentation to analyze the single cells in 3D using confocal microscopy, a technique that allows us to analyze cellular states in their native hydrated state.']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exclude (E), Check Full Text (C), or Include (I)?C\n",
      "Comments: whole cell reconstructions? sounds super relevant, but unclear from PMC figures\n",
      "Stop? (Y/N)N\n"
     ]
    }
   ],
   "source": [
    "# pull abstracts and sort into categories, save results in a dataframe\n",
    "# read abstract, type E, C, or I \n",
    "# \"Include\" criteria: whole-cell imaging including multiple organelles\n",
    "# comments: very brief, reason for decision, if \"C\" - what to look for; make note if prokaryotic system\n",
    "# must say \"Y\" to stop before moving on, otherwise it gets stuck in a loop and you may lose progress\n",
    "\n",
    "stage2_sort = pd.DataFrame(columns = ['Index', 'PMID', 'Exclude (E)/Check Full Text (C)/Include (I)', 'Comments'])\n",
    "stage2_sort\n",
    "index = 1 -1 # update first number after stopping and returning to where you left off, should start at PMID 36418108, always subtract by 1\n",
    "\n",
    "for item in range(len(coll_031523['PMID'])):\n",
    "    title = coll_031523['Title'][index]\n",
    "    print(coll_031523['PMID'][index],\"Title: \", title)\n",
    "\n",
    "    # some Pubmed entries lack an abstract, this would create an error otherwise\n",
    "    if 'Abstract' in stage1_fetch['PubmedArticle'][index]['MedlineCitation']['Article']:\n",
    "        abstract = stage1_fetch['PubmedArticle'][index]['MedlineCitation']['Article']['Abstract']['AbstractText']\n",
    "        print(\"Abstract: \", abstract)\n",
    "\n",
    "    else:\n",
    "        print(\"No abstract available, indicate Check Full Text (C)\")\n",
    "\n",
    "    sort_ECI = input('Exclude (E), Check Full Text (C), or Include (I)?')\n",
    "    while sort_ECI not in ['E', 'C', 'I']:\n",
    "        sort_ECI = print('whoops! invalid answer')\n",
    "        sort_ECI = input('Exclude (E), Check Full Text (C), or Include (I)?')\n",
    "\n",
    "    cmmts = input('Comments: ')\n",
    "#     comments.append(cmmts)\n",
    "\n",
    "#     print(stage2_sort)\n",
    "#     print(index,item,sort_ECI,cmmts)\n",
    "    stage2_sort.loc[len(stage2_sort)] = [index,coll_031523['PMID'][index],sort_ECI,cmmts]\n",
    "    \n",
    "    stop = input('Stop? (Y/N)')\n",
    "    if stop == 'Y':\n",
    "        break\n",
    "        \n",
    "    index+=1\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Index</th>\n",
       "      <th>PMID</th>\n",
       "      <th>Exclude (E)/Check Full Text (C)/Include (I)</th>\n",
       "      <th>Comments</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>35698308</td>\n",
       "      <td>C</td>\n",
       "      <td>whole cell?</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>36617247</td>\n",
       "      <td>C</td>\n",
       "      <td>3D?</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>36602467</td>\n",
       "      <td>E</td>\n",
       "      <td>bacteria; no relevant data</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>36702897</td>\n",
       "      <td>E</td>\n",
       "      <td>not whole cell</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>36774410</td>\n",
       "      <td>E</td>\n",
       "      <td>tissue scale; not whole-cell</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>83</td>\n",
       "      <td>38705628</td>\n",
       "      <td>C</td>\n",
       "      <td>relevant demo data?</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>84</td>\n",
       "      <td>38709484</td>\n",
       "      <td>E</td>\n",
       "      <td>focus on mitochondria; book chapter?</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>85</td>\n",
       "      <td>38756048</td>\n",
       "      <td>E</td>\n",
       "      <td>not whole cell, focus on pollenkitt (outer sub...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>86</td>\n",
       "      <td>38786091</td>\n",
       "      <td>I</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>87</td>\n",
       "      <td>38854505</td>\n",
       "      <td>C</td>\n",
       "      <td>whole cell reconstructions? sounds super relev...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>88 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   Index      PMID Exclude (E)/Check Full Text (C)/Include (I)  \\\n",
       "0      0  35698308                                           C   \n",
       "1      1  36617247                                           C   \n",
       "2      2  36602467                                           E   \n",
       "3      3  36702897                                           E   \n",
       "4      4  36774410                                           E   \n",
       "..   ...       ...                                         ...   \n",
       "83    83  38705628                                           C   \n",
       "84    84  38709484                                           E   \n",
       "85    85  38756048                                           E   \n",
       "86    86  38786091                                           I   \n",
       "87    87  38854505                                           C   \n",
       "\n",
       "                                             Comments  \n",
       "0                                         whole cell?  \n",
       "1                                                 3D?  \n",
       "2                          bacteria; no relevant data  \n",
       "3                                      not whole cell  \n",
       "4                        tissue scale; not whole-cell  \n",
       "..                                                ...  \n",
       "83                               relevant demo data?   \n",
       "84               focus on mitochondria; book chapter?  \n",
       "85  not whole cell, focus on pollenkitt (outer sub...  \n",
       "86                                                     \n",
       "87  whole cell reconstructions? sounds super relev...  \n",
       "\n",
       "[88 rows x 4 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "stage2_sort"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "# save results as excel file, indicate range of indices after underscore\n",
    "\n",
    "stage2_sort.to_excel('stage2_sort_021823-060324.xlsx')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.getcwd()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
